<!DOCTYPE html>
<html lang="en"> 
  <head>      
      <meta charset="UTF-8" />
      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
      <link rel="canonical" href="https://guerinpe.com/">
      <!-- Enable responsiveness on mobile devices-->
      <meta name="viewport" content="width=device-width, initial-scale=1" />

      <title> Pierre-Edouard Guerin | Bioinformatician </title>


      <link rel="shortcut icon" href="https:&#x2F;&#x2F;guerinpe.com/peguerin.ico" type="image/x-icon" />
      <!-- SEO -->
      <meta name="description" content="Scientific blog by bioinformatician Pierre-Edouard Guerin, covering  genetics, omics, engineering and machine learning.">
      <meta name="keywords" content="bioinformatics, computational biology, genetics, industry, management, omics, engineering, data science, biotechnology, machine learning, science, Pierre-Edouard Guerin">
      <meta name="author" content="Pierre-Edouard Guerin">

      <!-- Open Graph -->
      <meta property="og:title" content="Pierre-Edouard GUERIN" />
      <meta property="og:description" content="Science, omics, computational biology, bioinformatics, data science, genetics." />
      <meta property="og:image" content="https:&#x2F;&#x2F;guerinpe.com/avatar.png" />
      <meta property="og:image:alt" content="Portrait of Pierre-Edouard Guerin Bioinformatician Data Scientist" />
      <meta property="og:url" content="https://guerinpe.com" />
      <meta property="og:site_name" content="Pierre-Edouard GUERIN" />
      <meta property="og:type" content="website" />

      <!-- Twitter Card -->
      <meta name="twitter:title" content="Pierre-Edouard GUERIN" />
      <meta name="twitter:description" content="Science, omics, computational biology, bioinformatics, data science, genetics." />
      <meta name="twitter:image" content="https:&#x2F;&#x2F;guerinpe.com/avatar.png" />
      <meta name="twitter:card" content="summary" />
      <meta name="twitter:image:alt" content="scientific blog of Pierre-Edouard GUERIN" />
      
      <!-- schema markup SEO -->
      <script type="application/ld+json">
      {
        "@context": "https://schema.org",
        "@type": "Person",
        "name": "Pierre-Edouard Guerin",
        "url": "https://guerinpe.com",
        "image": "https://guerinpe.com/avatar.png",
        "jobTitle": "Bioinformatician Research Scientist",
        "worksFor": {
          "@type": "Organization",
          "name": "Florimond Desprez Group",
          "url": "https://www.florimond-desprez.com/fr/fr/groupe-florimond-desprez/notre-recherche.html"
        },
        "sameAs": [
          "https://www.linkedin.com/in/pierre-edouard-guerin",
          "https://twitter.com/guerin_pe",
          "https://github.com/Grelot",
          "https://gitlab.mbb.univ-montp2.fr/users/peguerin/groups",
          "https://www.cefe.cnrs.fr/fr/recherche/bc/bev/1048-desc/3594-pierre-edouard-guerin",
          "https://orcid.org/0000-0001-7909-3729",
          "https://www.researchgate.net/profile/Pierre-Edouard-Guerin",
          "https://www.kaggle.com/pierreedouardguerin",
          "https://scholar.google.com/citations?user=hj1ClrsAAAAJ&hl=en",
          "https://stackoverflow.com/users/11874491/pierre-edouard-guerin",
          "https://bioinfo-fr.net/user/peguerin"
        ],        
        "description": "Bioinformatician research scientist with focus on data engineering, genetics, and biotechnology."
      }
      </script>

      <!-- CSS -->
      <link rel="stylesheet" href="https://guerinpe.com/main.css" />      
      <link rel="preconnect" href="https://fonts.googleapis.com" />
      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
      <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=JetBrains+Mono:wght@400;600&family=Space+Grotesk:wght@600;700&display=swap" rel="stylesheet" />
      <!--FEEDS-->
      
        <link rel="alternate" type="application/rss+xml" title="RSS" href="https://guerinpe.com/atom.xml">
      

      <!--Javascript-->
      <script src="https://guerinpe.com/js/elasticlunr.min.js"></script>
      <script src="https://guerinpe.com/js/search.js"></script>
      <script src="https://guerinpe.com/js/active_nav.js"></script>
      <script src="https://guerinpe.com/js/share.js"></script>
      <script>
      window.MathJax = {
        tex: {
          inlineMath: [['$', '$'], ['\\(', '\\)']],
          displayMath: [['$$', '$$'], ['\\[', '\\]']]
        },
        svg: {
          fontCache: 'global'
        }
      };
      </script>
      <script id="MathJax-script" async
              src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js">
      </script>
      <!--mailchip-->



  </head>

  <body>
    
    <header class="header-container">
      
<div class="top-bar">

<div class="top-bar-left">
  <a href="https:&#x2F;&#x2F;guerinpe.com/" class="top-avatar">
    <img src="https:&#x2F;&#x2F;guerinpe.com/avatar.png" alt="Portrait of Pierre-Edouard Guerin Bioinformatician Data Scientist" />
  </a>
</div>
<div class="top-bar-center">
    <div class="site-title">
        <a href="https:&#x2F;&#x2F;guerinpe.com/">Pierre-Edouard Guerin</a>
</div>
  
</div>



    <div class="top-bar-right">
    <button class="top-icon-button" aria-label="Search" onclick="location.href='/search'">
        <!-- Search Icon -->
        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"
            fill="none" stroke="currentColor" stroke-width="2"
            stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search">
        <circle cx="11" cy="11" r="8"></circle>
        <path d="m21 21-4.3-4.3"></path>
        </svg>
    </button>

    <button class="top-icon-button" aria-label="Share" id="share-button">
        <!-- Share Icon -->
        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"
            fill="none" stroke="currentColor" stroke-width="2"
            stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-share">
        <path d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"></path>
        <polyline points="16 6 12 2 8 6"></polyline>
        <line x1="12" x2="12" y1="2" y2="15"></line>
        </svg>
    </button>

    <button class="top-contact-button">
       <a href="mailto:pierre-edouard.guerin@groupefd.com" role="button" aria-label="Send email to Pierre-Edouard Guerin">
  Contact
</a>
    </button>
    </div>
</div>


      

      <nav class="navbar">
        <a href="https://guerinpe.com/.">Home</a>             
        <a href="https://guerinpe.com/articles">Blog</a>
        <a href="https://guerinpe.com/resume">CV</a>
        <a href="https://guerinpe.com/activity">Activity</a>
        <a href="https://guerinpe.com/publications">Publications</a>
      </nav>               
    </header>    
    


    <br>
    
    
<div class="post">
       
  <h1 class="post-title">A Global Ocean Atlas of Eukaryotic Genes</h1>
    <br>

  <div class="post-info">
    Posted by <a href="https://guerinpe.com/.">Pierre-Edouard Guerin</a> · 7 min read · Published on September 19, 2020 
  </div>
  <hr>
  <p>Single-celled microeukaryotes and small multicellular zooplankton account for most of the planktonic biomass in the world’s ocean. Seawater samples were collected from the global ocean during the Tara Oceans expedition to generate a global ocean reference catalog of genes from planktonic eukaryotes sampled RNA and to explore the expression patterns of community at different microscopic scales with respect to biogeography and environmental conditions.</p>
<span id="continue-reading"></span><h2 id="rna-sequencing">RNA sequencing</h2>
<p>The key step in this technique is to convert RNA fragments into complementary DNA (cDNA library) using a reverse transcriptase. Adapters are then added to each end of the fragments. These adapters contain functional elements which permit sequencing. For example, the amplification element and the primary sequencing site. The cDNA <em>library</em> is then sequenced, producing short sequences called <em>reads</em> which correspond to either one or both ends of the fragment. The depth to which the library is sequenced varies depending on techniques which the output data will be used for.</p>
<h4 id="1-nucleic-acid-extraction">1. Nucleic acid extraction</h4>
<p>Cryogenic grinding is a very effective technique for taking hard substances, like plant and animal tissues, and turning them into dust. The DNA and RNA can be then extracted simulatenously. Then RNA is purified using a phenol-chloroform extraction method. DNA and RNA are quantified by a fluorometric method.</p>
<h4 id="2-library-construction">2. Library construction</h4>
<p>Deoxyribonuclease (DNase) treatment is applied on RNA extractions. To target coding RNA, 3' polyadenylated tail RNA selection is used to limit rRNA quantity. Then RNA is reverse transcribed to cDNA because DNA is more stable and to allow for amplification (which uses DNA polymerases). DNA is fragmented by sonification to obtain fragments of a predifined length of 101 bp. The cDNA for each experiment are indexed with a <em>barcode</em>, so that these experiments can be pooled into a single lane for multiplexed sequencing.</p>
<h4 id="3-sequencing">3. Sequencing</h4>
<p>Once the library is prepared, and adapters added, cDNA library is sequenced on HiSeq instrument with a read length of 101 bp in a paired-end mode.</p>
<div class="encart_inside_article">
<h2 id="key-words">Key words</h2>
<p><strong>Complementary DNA (cDNA):</strong> a DNA synthesized from a RNA molecule (<em>e.g.</em>, messenger RNA (mRNA)) in a reaction catalyzed by the enzyme reverse transcriptase.</p>
<p><strong>Library:</strong> a pool of DNA fragments with adapters attached. Adapters are designed to interact with the sequencing platform by the surface of the flow-cell (Illumina). A sequencing library can be made by starting from raw genomic DNA or from RNA. In any RNA sequencing library there’s an additional step: the RNA conversion in cDNA. The fragmentation step can be done before or after the cDNA synthesis.</p>
<p><strong>Reads:</strong>  a literal sequence generated by the sequencing instrument. The read is ultimately the sequence of a section of a unique fragment from the library. If an RNA is expressed in high copies then there will be more reads coming from it so that reads can be redundant as well.</p>
<p><strong>Multiplexing:</strong> to pool and sequence libraries simultaneously during a single run on sequencing instruments. With multiplex sequencing, individual <em>barcode</em> sequences are added to each DNA fragment during next-generation sequencing (NGS) library preparation so that each <em>read</em> can be identified and sorted before the final data analysis.</p>
</div>
<h2 id="assembly-and-gene-catalog-construction">Assembly and gene catalog construction</h2>
<p>Each read sequences are assembled into a contig using <a href="https://www.ebi.ac.uk/~zerbino/velvet/">VELVET</a>. VELVET is an assembler which takes advantage of De Bruijn graph approach to focus on observed words (or <em>k-mers</em>) among the read sequences. A given <em>k-mer</em> is therefore represented  by a unique node, regardless of how many times it was observed. The costliest part of constructing a de Bruijn graph consists in hashing all the reads, according to a given word length (selected <em>k-mer</em> size is 63 in this study). This operation offers nonetheless an advantageous time complexity compared to a  general pairwise alignment of all the sequences, especially given the high coverage depths encountered. Once all the reads have been hashed, each of their paths are traced along the k-mer nodes, incrementing coverage and creating the appropriate arcs along the way.</p>
<p>For each cluster of contig, the longest contig is kept as the representative sequence of the cluster. Sequences were BLASTed against a reference database of known sequences from Prokatyote, mitochondrial and chloroplastic specimens. Sequence with at least 70% identity with the reference database are removed. So that we obtain a catalog of <strong>unigenes</strong>. In summary a unigene is a complete or partial transcript assembled from metatranscriptomic reads of at least one <em>Tara</em> Oceans station.</p>
<h2 id="taxonomic-assignment">Taxonomic assignment</h2>
<p>To assign a taxonomic group to each unigene, a reference database is built from <strong>UniRef</strong>. Sequence similarities between metatranscriptomic gene catalog and reference database is handled by <a href="https://github.com/bbuchfink/diamond">DIAMOND</a>. DIAMOND is a sequence aligner for protein and translated DNA searches, designed for high performance analysis of big sequence data. It allows pairwise alignment of proteins and translated DNA at 100x-20,000x speed of BLAST. Taxonomic affiliation uses a weighted lowest common ancestor approach. For each unigene only taxon reference sequences with at least 90% similarity were kept.</p>
<h2 id="functionnal-characterization-of-unigenes">Functionnal characterization of unigenes</h2>
<p>Protein <em>domain</em> prediction is performed using <a href="http://hmmer.org/">HMMER</a>. HMMER is used for searching sequence databases for sequence homologs, and for making sequence alignments. Unigenes were seeked in the profile database <strong>Pfam</strong>. So that a protein domain is assigned to each unigene. Unigenes without assigned protein domain are not kept.</p>
<div class="encart_inside_article">
<h2 id="protein-and-profile-database">Protein and profile database</h2>
<p><strong>UniProt:</strong> the database of protein sequence and functional information, many entries being derived from genome sequencing projects.</p>
<p><strong>UniGene:</strong> partition of sequences into a non-redundant set of gene-oriented clusters. Each UniGene cluster contains sequences that represent a unique gene, as well as related information such as the tissue types in which the gene has been expressed and map location.</p>
<p><strong>UniRef:</strong> <strong>Uni</strong>Prot <strong>Ref</strong>erence Clusters provide clustered sets of sequences from UniProt in order to obtain complete coverage of the <a href="https://en.wikipedia.org/wiki/Sequence_space_(evolution)">sequence space</a> at several resolutions.</p>
<p><strong>Pfam:</strong> the database of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs). Proteins are generally composed of one or more functional regions, commonly termed <em>domains</em>. Different combinations of domains give rise to the diverse range of proteins found in nature. Pfam entry data are based on UniProt reference proteomes.</p>
</div>
<h2 id="expression-and-abundance-of-unigenes">Expression and abundance of unigenes</h2>
<p>In order to estimate the abundance (a proxy of expression) of each unigene in each sample, reads were mapped against the gene catalog. Unigene expression values and genomic occurences are computed in <strong>RPKM</strong>. Then the abundance of each unigene was normalized and formulated in two different ways.</p>
<ul>
<li>the gene abundance relative to the abundance of all genes from the same taxon in percentage.</li>
<li>the fraction of the gene abundance attributed to a particular taxonomic group.</li>
</ul>
<div style="background: #ecf0f1 ;">
<p><strong>Reads Per Kilobase of transcript, per Million mapped reads (RPKM)</strong> is a normalized unit of transcript expression. It scales by transcript length to compensate for the fact that most RNA-seq protocols will generate more sequencing reads from longer RNA molecules</p>
</div>
<h2 id="conclusion">Conclusion</h2>
<p>The global ocean transcript catalog reported here represents a first resource to study extensively and uniformly the gene content of eukaryotes and the dynamics of their expression in the environment. Metatranscriptomics is an effective approach to observe functionnal ecology of planktonic communities in the ocean.</p>
<h2 id="references">References</h2>
<blockquote>
<p><strong>A global ocean atlas of eukaryotic genes</strong></p>
<p><em>Quentin Carradec, Eric Pelletier, [...] Patrick Wincker</em></p>
<p>Nature Communications. 2018 Jan 25. DOI: <a href="https://doi.org/10.1038/s41467-017-02342-1">10.1038/s41467-017-02342-1</a></p>
</blockquote>

  <br>
  <br>
  <br>
  <br>
  <hr>
  
<div class="post-content-wrapper">
    <div class="post-info">
    Published on September 19, 2020
    <br><br>
    </div>
    
    <hr class="bold">     
    <div class="home-latest-articles">
        <h2 class="home-section-title">Relevant Tags</h2>
        <div class="tags-article">
            <ul>
                 
                
                <li style="background: #16A664">
                <a style="color: #ffffff" 
                   href="https://guerinpe.com/tags/omics/">omics</a>&nbsp;
                </li> 
                
                <li style="background: #FB4C2F">
                <a style="color: #ffffff" 
                   href="https://guerinpe.com/tags/sciences/">sciences</a>&nbsp;
                </li> 
                
                        
            </ul>    
        </div>
    </div>

    <div class="home-latest-articles">
        <h2 class="home-section-title">About the Author</h2>

        <div class="about-author">
            <div class="author-container">
                <img src="https:&#x2F;&#x2F;guerinpe.com/avatar.png" alt="Portrait of Pierre-Edouard Guerin Bioinformatician Data Scientist"/>
                <div class="author-info">
                    <a href="https:&#x2F;&#x2F;guerinpe.com" class="author-name">Pierre-Edouard Guerin</a>
                    <div class="author-description">Bioinformatician</div>
                    <div class="brand_nav">
                        
                        
                        <a href="https:&#x2F;&#x2F;www.linkedin.com&#x2F;in&#x2F;pierre-edouard-guerin"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M20.447 20.452h-3.554v-5.569c0-1.328-.027-3.037-1.852-3.037-1.853 0-2.136 1.445-2.136 2.939v5.667H9.351V9h3.414v1.561h.046c.477-.9 1.637-1.85 3.37-1.85 3.601 0 4.267 2.37 4.267 5.455v6.286zM5.337 7.433c-1.144 0-2.063-.926-2.063-2.065 0-1.138.92-2.063 2.063-2.063 1.14 0 2.064.925 2.064 2.063 0 1.139-.925 2.065-2.064 2.065zm1.782 13.019H3.555V9h3.564v11.452zM22.225 0H1.771C.792 0 0 .774 0 1.729v20.542C0 23.227.792 24 1.771 24h20.451C23.2 24 24 23.227 24 22.271V1.729C24 .774 23.2 0 22.222 0h.003z"/></svg> </a>
                        
                        <a href="https:&#x2F;&#x2F;twitter.com&#x2F;guerin_pe"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.954 4.569c-.885.389-1.83.654-2.825.775 1.014-.611 1.794-1.574 2.163-2.723-.951.555-2.005.959-3.127 1.184-.896-.959-2.173-1.559-3.591-1.559-2.717 0-4.92 2.203-4.92 4.917 0 .39.045.765.127 1.124C7.691 8.094 4.066 6.13 1.64 3.161c-.427.722-.666 1.561-.666 2.475 0 1.71.87 3.213 2.188 4.096-.807-.026-1.566-.248-2.228-.616v.061c0 2.385 1.693 4.374 3.946 4.827-.413.111-.849.171-1.296.171-.314 0-.615-.03-.916-.086.631 1.953 2.445 3.377 4.604 3.417-1.68 1.319-3.809 2.105-6.102 2.105-.39 0-.779-.023-1.17-.067 2.189 1.394 4.768 2.209 7.557 2.209 9.054 0 13.999-7.496 13.999-13.986 0-.209 0-.42-.015-.63.961-.689 1.8-1.56 2.46-2.548l-.047-.02z"/></svg> </a>
                        
                        <a href="https:&#x2F;&#x2F;github.com&#x2F;Grelot"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/></svg> </a>
                        
                        <a href="https:&#x2F;&#x2F;gitlab.mbb.univ-montp2.fr&#x2F;users&#x2F;peguerin&#x2F;groups"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.955 13.587l-1.342-4.135-2.664-8.189c-.135-.423-.73-.423-.867 0L16.418 9.45H7.582L4.919 1.263C4.783.84 4.185.84 4.05 1.26L1.386 9.449.044 13.587c-.121.375.014.789.331 1.023L12 23.054l11.625-8.443c.318-.235.453-.647.33-1.024"/></svg> </a>
                        
                        
                    </div>
                </div>
            </div>
        </div>
    </div>

    


<div class="home-latest-articles">
  <h2 class="home-section-title">Latest Articles</h2>

 <ul class="home-article-list">
  
    <li class="home-article-item">
      <a href="https:&#x2F;&#x2F;guerinpe.com&#x2F;articles&#x2F;research-tax-credit&#x2F;" class="home-article-link">
        <h3 class="home-article-title">Research Tax Credit</h3>
        <div class="home-article-summary">The Research Activities Tax Credit is a tax credit that incentivizes private companies to increase their Research and Development (R&amp;D).  Within my company, I have been tasked with writing the Research Tax Credit (CIR) justification report for France. Here, the method for writing such a report.</div>
        <div class="home-article-date">  OCT 2025 · PIERRE-EDOUARD GUERIN </div>
      </a>
      
    </li>
  
    <li class="home-article-item">
      <a href="https:&#x2F;&#x2F;guerinpe.com&#x2F;articles&#x2F;turing-complete-from-logical-gates-to-cpu-architecture&#x2F;" class="home-article-link">
        <h3 class="home-article-title">Turing Complete: From Logical Gates to CPU Architecture</h3>
        <div class="home-article-summary">In 2021, LevelHead published Turing Complete, a game about computer science. My friend Christophe Georgescu recommended me to play it. Unfortunately, I took his advice and now I can not stop to play this game! The game challenges you to design an entire computer from scratch. You start with basic logic gates, then move on to components, memory, CPU architecture, and finally assembly programming. By the way, the game is neat and present all these concepts in a playful and intuitive way.</div>
        <div class="home-article-date">  SEP 2025 · PIERRE-EDOUARD GUERIN </div>
      </a>
      
    </li>
  
    <li class="home-article-item">
      <a href="https:&#x2F;&#x2F;guerinpe.com&#x2F;articles&#x2F;how-to-manage-project&#x2F;" class="home-article-link">
        <h3 class="home-article-title">How to Manage a Project?</h3>
        <div class="home-article-summary">In any company, every task is part of a project. I am responsible for managing multiple projects each year. I have to present deliverables to stakeholders, meet deadlines, allocate mandays and coordinate everyone’s actions. This is a meticulous work that requires a strong methodology.</div>
        <div class="home-article-date">  JUN 2025 · PIERRE-EDOUARD GUERIN </div>
      </a>
      
    </li>
  
</ul>


  <div class="home-see-all-container">
    <button class="home-see-all-button ">
    <a href="https:&#x2F;&#x2F;guerinpe.com/articles" role="button">See all &gt;</a>
    </button>
  </div>
</div>
</div>

</div>


    <br>

  <footer class="footer">
  <h2>Site Plan</h2>
  <div class="footer-inner">
    <div class="footer-links">
      <a href="https://guerinpe.com/.">Home</a>
      <a href="https://guerinpe.com/articles">Blog</a>
      <a href="https://guerinpe.com/resume/">CV</a>
      <a href="https://guerinpe.com/activity">Activity</a>
      <a href="https://guerinpe.com/publications">Publications</a>
      <a href="https://guerinpe.com/search">Search</a>
    </div>
</div>

  <h2>Tags</h2>


  
  <mark>Biotech</mark>
  
  <mark>Career</mark>
  
  <mark>Communication</mark>
  
  <mark>Computer-science</mark>
  
  <mark>Data-engineering</mark>
  
  <mark>Data-science</mark>
  
  <mark>Management</mark>
  
  <mark>Math</mark>
  
  <mark>Med-science</mark>
  
  <mark>Omics</mark>
  
  <mark>Plant-science</mark>
  
  <mark>Sciences</mark>
  
  <mark>Society</mark>
  
  <mark>Software-engineering</mark>
  
  <mark>Sys</mark>
  
  <mark>Tech</mark>
  
  <mark>Zoo</mark>
    


  <div class="footer-inner">

    <h2>Social Networks</h2>

    <div class="footer-social">
        
        
        <a href="https:&#x2F;&#x2F;www.linkedin.com&#x2F;in&#x2F;pierre-edouard-guerin" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M20.447 20.452h-3.554v-5.569c0-1.328-.027-3.037-1.852-3.037-1.853 0-2.136 1.445-2.136 2.939v5.667H9.351V9h3.414v1.561h.046c.477-.9 1.637-1.85 3.37-1.85 3.601 0 4.267 2.37 4.267 5.455v6.286zM5.337 7.433c-1.144 0-2.063-.926-2.063-2.065 0-1.138.92-2.063 2.063-2.063 1.14 0 2.064.925 2.064 2.063 0 1.139-.925 2.065-2.064 2.065zm1.782 13.019H3.555V9h3.564v11.452zM22.225 0H1.771C.792 0 0 .774 0 1.729v20.542C0 23.227.792 24 1.771 24h20.451C23.2 24 24 23.227 24 22.271V1.729C24 .774 23.2 0 22.222 0h.003z"/></svg>
        </a>
      
        <a href="https:&#x2F;&#x2F;twitter.com&#x2F;guerin_pe" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.954 4.569c-.885.389-1.83.654-2.825.775 1.014-.611 1.794-1.574 2.163-2.723-.951.555-2.005.959-3.127 1.184-.896-.959-2.173-1.559-3.591-1.559-2.717 0-4.92 2.203-4.92 4.917 0 .39.045.765.127 1.124C7.691 8.094 4.066 6.13 1.64 3.161c-.427.722-.666 1.561-.666 2.475 0 1.71.87 3.213 2.188 4.096-.807-.026-1.566-.248-2.228-.616v.061c0 2.385 1.693 4.374 3.946 4.827-.413.111-.849.171-1.296.171-.314 0-.615-.03-.916-.086.631 1.953 2.445 3.377 4.604 3.417-1.68 1.319-3.809 2.105-6.102 2.105-.39 0-.779-.023-1.17-.067 2.189 1.394 4.768 2.209 7.557 2.209 9.054 0 13.999-7.496 13.999-13.986 0-.209 0-.42-.015-.63.961-.689 1.8-1.56 2.46-2.548l-.047-.02z"/></svg>
        </a>
      
        <a href="https:&#x2F;&#x2F;github.com&#x2F;Grelot" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/></svg>
        </a>
      
        <a href="https:&#x2F;&#x2F;gitlab.mbb.univ-montp2.fr&#x2F;users&#x2F;peguerin&#x2F;groups" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.955 13.587l-1.342-4.135-2.664-8.189c-.135-.423-.73-.423-.867 0L16.418 9.45H7.582L4.919 1.263C4.783.84 4.185.84 4.05 1.26L1.386 9.449.044 13.587c-.121.375.014.789.331 1.023L12 23.054l11.625-8.443c.318-.235.453-.647.33-1.024"/></svg>
        </a>
      
      
    </div>
  </div>
</div>
  
  


  <h2>Site Information</h2>

  <div class="footer-meta">
    <p>
     2015-2025
     guerinpe.com by Pierre-Edouard GUERIN is under a 
      <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a> ·
      <a href="https://www.getzola.org/">Powered by Zola</a> ·
      <a href="https://guerinpe.com/atom.xml">RSS feed</a>
    </p>
  </div>
</footer>

  </body>



</html>
