<html lang=""> 
  <head>
      <title> {{config.title}} </title>
      <meta http-equiv="X-UA-Compatible" content="IE=edge">
      <meta http-equiv="content-type" content="text/html; charset=utf-8">      
      <!-- Enable responsiveness on mobile devices-->
      <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=6">
      <!-- CSS -->
       <link rel="stylesheet" href="{{ get_url(path="print.css", trailing_slash=false) }}" media="print">
       <link rel="stylesheet" href="{{ get_url(path="corps.css", trailing_slash=false) }}">
       <link rel="stylesheet" href="{{ get_url(path="sobre.css", trailing_slash=false) }}">
       <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700|Abril+Fatface">

  </head>
    <body>
    {% block sidebar %}   
    <div class="sidebar">       
      <a href="{{config.base_url}}"><span class="avatar" style="background-image:url('{{config.base_url}}/avatar.png')"></span></a>
      <div class="sidebar-name"><p> PIERRE-EDOUARD <b>GUERIN</b> </p></div>
       <div class="sidebar-description"><p> {{config.description}} </p></div>
      
      <div class="sidebar-nav-brand">
          {% block sidebar_nav_brand %}
          {% for link in config.extra.links %}
          <a href="{{link.url}}" style="content: url({{config.base_url}}/brands/{{link.name}}.svg)">{{link.name}}</a><b> · </b>
          {% endfor %}
          <a href={{config.extra.myjob_site}}>{{config.extra.myjob_name}}</a>
          {% endblock sidebar_nav_brand %}
      </div>

      <hr>      
        <div class="sidebar-about">
            {% block sidebar_about %}                    
              {% if config.description %}                    
                <p class="lead"><a href="{{ get_url(path="articles") }}"> Blog </a></p>
                <p class="lead"><a href="{{ get_url(path="resume") }}/#"> Curriculum Vitae </a></p>
              {% endif %}
            {% endblock sidebar_about %}
        </div>
        <hr>
     <ul class="sidebar-nav">
                {% block sidebar_nav_infos %}
               
                          <li class="sidebar-nav-item"><a href="{{ get_url(path="resume") }}/#projects">Projects</a></li>
                          <li class="sidebar-nav-item"><a href=" {{ get_url(path="resume") }}/#softwares">Softwares & publications</a></li>
                          <li class="sidebar-nav-item"><a href="https://github.com/Grelot/machine_learning_demo#machine_learning_demo" style="color:orange">Machine learning</a></li>

                           
               
                {% endblock sidebar_nav_infos %}                    
            </ul> 

    </div>

    {% endblock sidebar %}
    <div class="content container">    

      {% block content %}
      <div>
      <br>

     
      <h1> About</h1>
      
      <p>After a Master thesis in bioinformatics, during which I developed my first machine learning softwares applied to 3D proteomics 
        a technique that couples chemical cross-linking with mass spectrometry and has emerged as a tool to study protein conformations,
        I join INSERM Genetics of Diabetes research unit for developing softwares 
        to automate detection of genetic variants linked to the diabete phenotype onto human genomes.
      </p>

      <p>As I was already developing my own tools to deal with sequencing and "omics" data, I followed a course of statitics about population genetics.
        It aims me to be recruited for the european project RESERVEBENEFIT in 2017.
      </p>
    
      <p>Within this project, I learn to handle HPC platforms and to ease scientific reproduction.
         I take advange of Git, Singularity, Docker, Conda, Jupyter notebook, SnakeMake or Nextflow.
        I successfully performed new genome DNA sequence assemblies for 3 species and variant calling of 1600 individuals.
      </p>
      <p>
        In parallel to this project, I joined SEACONNECT project to process RADseq data and carry out population genetics analysis. 
        I learn to handle geographical and environmental data from public database.
      </p>
      <p> In 2019, I integrate CNRS of Montpellier as a computational biologist and contribute to various projects
         in collaboration with different teams on metabarcoding and metagenomics.
         I also create automated workflows and database system for storing, retrieving and processing metabarcoding data for the MONACO MARINE WORLDWIDE EXPEDITION.
      </p>

      <p>
      My main computational languages are Bash, Python and R. However I also realised several projects using C, C++, Perl or Julia.
      See my <a href="{{ get_url(path="resume") }}">Curriculum Vitae</a> for more details.
     </p>
     <br>
     <br>
    </div>

    {% endblock content %}
    </div>
 
    <div class="footer"> 
      {{config.extra.myjob_address}} {{config.extra.mytown}}, {{config.extra.mycountry}}<br>
      <a href="mailto:{{config.extra.webmail}}"> {{config.extra.webmail | replace(from="@", to=" [at] ") }}</a> {{config.extra.myjob_tel}}
      <br><br><br><br>
      <small> Copyright © 2019-2020 Pierre-Edouard GUERIN. All Rights Reserved | Powered by ZOLA</small>
    </div>
    </body>
</html>
