<!DOCTYPE html>
<html lang="en"> 
  <head>      
      <meta charset="UTF-8" />
      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
      <link rel="canonical" href="https://guerinpe.com/">
      <!-- Enable responsiveness on mobile devices-->
      <meta name="viewport" content="width=device-width, initial-scale=1" />

      <title> Pierre-Edouard Guerin | Bioinformatician </title>


      <link rel="shortcut icon" href="https:&#x2F;&#x2F;guerinpe.com/peguerin.ico" type="image/x-icon" />
      <!-- SEO -->
      <meta name="description" content="Scientific blog by bioinformatician Pierre-Edouard Guerin, covering  genetics, omics, engineering and machine learning.">
      <meta name="keywords" content="bioinformatics, computational biology, genetics, industry, management, omics, engineering, data science, biotechnology, machine learning, science, Pierre-Edouard Guerin">
      <meta name="author" content="Pierre-Edouard Guerin">

      <!-- Open Graph -->
      <meta property="og:title" content="Pierre-Edouard GUERIN" />
      <meta property="og:description" content="Science, omics, computational biology, bioinformatics, data science, genetics." />
      <meta property="og:image" content="https:&#x2F;&#x2F;guerinpe.com/avatar.png" />
      <meta property="og:image:alt" content="Portrait of Pierre-Edouard Guerin Bioinformatician Data Scientist" />
      <meta property="og:url" content="https://guerinpe.com" />
      <meta property="og:site_name" content="Pierre-Edouard GUERIN" />
      <meta property="og:type" content="website" />

      <!-- Twitter Card -->
      <meta name="twitter:title" content="Pierre-Edouard GUERIN" />
      <meta name="twitter:description" content="Science, omics, computational biology, bioinformatics, data science, genetics." />
      <meta name="twitter:image" content="https:&#x2F;&#x2F;guerinpe.com/avatar.png" />
      <meta name="twitter:card" content="summary" />
      <meta name="twitter:image:alt" content="scientific blog of Pierre-Edouard GUERIN" />
      
      <!-- schema markup SEO -->
      <script type="application/ld+json">
      {
        "@context": "https://schema.org",
        "@type": "Person",
        "name": "Pierre-Edouard Guerin",
        "url": "https://guerinpe.com",
        "image": "https://guerinpe.com/avatar.png",
        "jobTitle": "Bioinformatician Research Scientist",
        "worksFor": {
          "@type": "Organization",
          "name": "Florimond Desprez Group",
          "url": "https://www.florimond-desprez.com/fr/fr/groupe-florimond-desprez/notre-recherche.html"
        },
        "sameAs": [
          "https://www.linkedin.com/in/pierre-edouard-guerin",
          "https://twitter.com/guerin_pe",
          "https://github.com/Grelot",
          "https://gitlab.mbb.univ-montp2.fr/users/peguerin/groups",
          "https://www.cefe.cnrs.fr/fr/recherche/bc/bev/1048-desc/3594-pierre-edouard-guerin",
          "https://orcid.org/0000-0001-7909-3729",
          "https://www.researchgate.net/profile/Pierre-Edouard-Guerin",
          "https://www.kaggle.com/pierreedouardguerin",
          "https://scholar.google.com/citations?user=hj1ClrsAAAAJ&hl=en",
          "https://stackoverflow.com/users/11874491/pierre-edouard-guerin",
          "https://bioinfo-fr.net/user/peguerin"
        ],        
        "description": "Bioinformatician research scientist with focus on data engineering, genetics, and biotechnology."
      }
      </script>

      <!-- CSS -->
      <link rel="stylesheet" href="https://guerinpe.com/main.css" />      
      <link rel="preconnect" href="https://fonts.googleapis.com" />
      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
      <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=JetBrains+Mono:wght@400;600&family=Space+Grotesk:wght@600;700&display=swap" rel="stylesheet" />
      <!--FEEDS-->
      
        <link rel="alternate" type="application/rss+xml" title="RSS" href="https://guerinpe.com/atom.xml">
      

      <!--Javascript-->
      <script src="https://guerinpe.com/js/elasticlunr.min.js"></script>
      <script src="https://guerinpe.com/js/search.js"></script>
      <script src="https://guerinpe.com/js/active_nav.js"></script>
      <script src="https://guerinpe.com/js/share.js"></script>
      <script>
      window.MathJax = {
        tex: {
          inlineMath: [['$', '$'], ['\\(', '\\)']],
          displayMath: [['$$', '$$'], ['\\[', '\\]']]
        },
        svg: {
          fontCache: 'global'
        }
      };
      </script>
      <script id="MathJax-script" async
              src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js">
      </script>
      <!--mailchip-->



  </head>

  <body>
    
    <header class="header-container">
      
<div class="top-bar">

<div class="top-bar-left">
  <a href="https:&#x2F;&#x2F;guerinpe.com/" class="top-avatar">
    <img src="https:&#x2F;&#x2F;guerinpe.com/avatar.png" alt="Portrait of Pierre-Edouard Guerin Bioinformatician Data Scientist" />
  </a>
</div>
<div class="top-bar-center">
    <div class="site-title">
        <a href="https:&#x2F;&#x2F;guerinpe.com/">Pierre-Edouard Guerin</a>
</div>
  
</div>



    <div class="top-bar-right">
    <button class="top-icon-button" aria-label="Search" onclick="location.href='/search'">
        <!-- Search Icon -->
        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"
            fill="none" stroke="currentColor" stroke-width="2"
            stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-search">
        <circle cx="11" cy="11" r="8"></circle>
        <path d="m21 21-4.3-4.3"></path>
        </svg>
    </button>

    <button class="top-icon-button" aria-label="Share" id="share-button">
        <!-- Share Icon -->
        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 24 24"
            fill="none" stroke="currentColor" stroke-width="2"
            stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-share">
        <path d="M4 12v8a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2v-8"></path>
        <polyline points="16 6 12 2 8 6"></polyline>
        <line x1="12" x2="12" y1="2" y2="15"></line>
        </svg>
    </button>

    <button class="top-contact-button">
       <a href="mailto:pierre-edouard.guerin@groupefd.com" role="button" aria-label="Send email to Pierre-Edouard Guerin">
  Contact
</a>
    </button>
    </div>
</div>


      

      <nav class="navbar">
        <a href="https://guerinpe.com/.">Home</a>             
        <a href="https://guerinpe.com/articles">Blog</a>
        <a href="https://guerinpe.com/resume">CV</a>
        <a href="https://guerinpe.com/activity">Activity</a>
        <a href="https://guerinpe.com/publications">Publications</a>
      </nav>               
    </header>    
    


    <br>
    
    
<div class="post">
       
  <h1 class="post-title">Multi Dimensional Scaling</h1>
    <br>

  <div class="post-info">
    Posted by <a href="https://guerinpe.com/.">Pierre-Edouard Guerin</a> · 5 min read · Published on April 3, 2022 
  </div>
  <hr>
  <p>Multi Dimensional Scaling (MDS) is very similar to <a href="/articles/principal-component-analysis/">Principal Component Analysis (PCA)</a>, except that instead of converting correlations into a 2D graph, it converts distances among the samples into a 2D graph.</p>
<span id="continue-reading"></span><h2 id="definition">Definition</h2>
<p>Imagine that you have ten runners and their measured performance for ten races. You want to visualize on a 2D graph the similarity (or dissimilarity) of the performances of the different runners. <strong>Multi Dimensional Scaling</strong> is a method to compute coordinates for each item in a chosen number of dimensions (usually 2D) so that the distances between points match the dissimilarities between items as closely as possible.</p>
<ul>
<li>In other word we calculate for each ${\binom{10}{2} = 45}$ pairs of runners, a distance or dissimilarity based on their results on each race. The results are stored in a <strong>dissimilarity matrix</strong>.</li>
<li>These distances are then computed using a stress function that minimize the difference between interpoint distance on the graph and the actual  distance calculated in the dissimilarity matrix.</li>
<li>We plot the 2D coordinates: runners with similar performance end up close and those with different performance end up far. Note that MDS can also be used on greater dimension (3D, 4D, ...).</li>
</ul>
<p>Let's calculate <strong>Euclidian distance</strong> for our dissimilarity matrix:</p>
<p>$$
\text{Euclidean Distance}(v, w) = \sqrt{\sum_{i=1}^{n} (v_i - w_i)^2}
$$</p>
<ul>
<li>${(v, w)}$ pair of two items ${v}$ and ${w}$ (<em>e.g</em> two runners we compare)</li>
<li>${i}$ the values to compare (<em>e.g</em> the measured performance ${v_i}$ and ${w_i}$ for runners ${v}$ and ${w}$ for the race ${i}$)</li>
</ul>
<p>Then MDS minimize the calculated distances for each pair in order to plot them together on a 2D graph:</p>
<ul>
<li>Given distances ${d_{ij}}$ between any runners ${i}$ and ${j}$</li>
<li>Find positions ${(x_{ij}, y_{ij})}$</li>
<li>With 2D space ${k = 2}$</li>
<li>That minimize a stress function</li>
</ul>
<p>$$
\sum_{k=1}^{k=2}\sum_{i}\sum_{j} (|x_{ik}-x_{jk}| - d_{ij})^2
$$</p>
<p>⚠️ When the dissimilarities are Euclidean distances, MDS gives the same coordinates of points as PCA. This is because minimizing the Euclidean distances between points is equivalent to maximizing their linear correlations.</p>
<p>There are all distances you can compute with <code>rdist</code> in R:</p>
<p><strong>Manhattan distance</strong></p>
<p>$$
d(x,y) = \sum_{i=1} |x_i - y_i|
$$</p>
<ul>
<li>Counts the total absolute differences.</li>
<li>Useful when you care about additive differences <em>e.g</em> distance between genotypes</li>
</ul>
<p><strong>Canberra distance</strong></p>
<p>$$
d(x,y) = \sum_{i=1} \frac{|x_i - y_i|}{|x_i| + |y_i|}
$$</p>
<ul>
<li>Normalizes the difference by the magnitude of the values.</li>
<li>Sensitive to small change near zero</li>
<li>Useful for data with many zeros <em>e.g.</em> allele frequencies in populations.</li>
</ul>
<p><strong>Binary distance (Jaccard)</strong></p>
<p>$$
d(x,y) = \frac{r + s}{q + r + s}
$$
<small></p>
<ul>
<li>Where ${q}$ number of cases where ${(x = 1 ; y = 1)}$</li>
<li>Where ${r}$ number of cases where ${(x = 1 ; y = 0)}$</li>
<li>Where ${s}$ number of cases where ${(x = 0 ; y = 1)}$
</small></li>
</ul>
<ul>
<li>Measure dissimilarity between presence/absence data</li>
<li>Ignore absence/absence case</li>
<li>Useful for presence/absence data <em>e.g.</em> grid of regions with observed species or not within</li>
</ul>
<p><strong>Maximum distance (Chebyshev)</strong></p>
<p>$$
d(x,y) = \max_{i} |x_i - y_i|
$$</p>
<ul>
<li>Only capture largest single difference</li>
<li>Useful if the worst-case difference dominates <em>e.g.</em> nucleotide pair read quality control</li>
</ul>
<h2 id="example-of-distance-between-genotypes">Example of distance between genotypes</h2>
<h3 id="data-set">Data set</h3>
<p>We work on a collection of 10 tetraploid genotypes and 10,000 SNPs. We want to visualize how close the genotypes are on a 2D graph.</p>
<pre><code>vcf &lt;- read_tsv(file=&quot;example.vcf&quot;)
</code></pre>
<table><thead><tr><th>SNP_ID</th><th>A</th><th>B</th><th>C</th><th>D</th><th>E</th><th>F</th><th>G</th><th>H</th><th>I</th><th>J</th></tr></thead><tbody>
<tr><td>SNP00</td><td>0</td><td>1</td><td>0</td><td>0</td><td>1</td><td>1</td><td>1</td><td>0</td><td>0</td><td>0</td></tr>
<tr><td>SNP01</td><td>4</td><td>4</td><td>0</td><td>0</td><td>1</td><td>0</td><td>1</td><td>0</td><td>4</td><td>4</td></tr>
<tr><td>SNP02</td><td>0</td><td>2</td><td>3</td><td>2</td><td>2</td><td>3</td><td>2</td><td>3</td><td>0</td><td>0</td></tr>
<tr><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td><td>...</td></tr>
<tr><td>SNP9999</td><td>2</td><td>0</td><td>4</td><td>4</td><td>1</td><td>4</td><td>1</td><td>4</td><td>2</td><td>2</td></tr>
</tbody></table>
<h3 id="dissimilarity-matrix">Dissimilarity Matrix</h3>
<p>A <strong>dissimilarity matrix</strong> is a table that shows the distance between pairs of vectors. Here our vectors are genotypes observed on 10,000 SNPs. We use Manhattan measure to compute distances between genotypes. We observe 5 classes of genotypes 0,1,2,3,4. For a given pair ${(v, w)}$ of observed genotypes ${i}$, the maximum distance is 4 - 0 = 4 and the minimum distance is 0.</p>
<p>$$
\text{Manathan Distance}:  \sum_{i} |v_i - w_i|
$$</p>
<p>As we cannot process missing data, we remove SNPs with missing calls so that we keep 9,000/10,000 SNPs to generate the dissimilarity matrix.</p>
<p>In R, you need <code>rdist</code> function from the <a href="https://cran.r-project.org/web/packages/rdist/">rdist R package</a> to compute pairwise distances.</p>
<pre><code>vcfDist &lt;- rdist::rdist(t(vcf[,-1] %&gt;% na.omit) , metric= &quot;manhattan&quot;)
dst &lt;- data.matrix(vcfDist)
colorsLevels =hcl.colors(100, &quot;Inferno&quot;)
dimDst &lt;- ncol(dst)
par(mar = c(5, 5, 5, 5))
fields::image.plot(1:dimDst, 1:dimDst, dst, axes = FALSE, xlab=&quot;&quot;, ylab=&quot;&quot;, col=colorsLevels)
axis(1, 1:dimDst, row.names(dst), cex.axis = 0.7, las=3)
axis(2, 1:dimDst, colnames(dst), cex.axis = 0.7, las=1)
</code></pre>
<p>This is a plot of the pairwise manhattan distance between genotypes. The lighter the greater the distance.</p>
<p><img src="https://guerinpe.com/articles/multi-dimensional-scaling/./example_plot_dissimilarity_matrix.jpg" alt="example plot dissimilarity matrix" /></p>
<h3 id="multi-dimensional-scaling">Multi dimensional scaling</h3>
<p>We want to visualize genotypes distance in the dissimilarity matrices as a set of geometry points. Here the
term <em>distance</em> is standing for <em>measure of dissimilarity</em>. We want to project our manhattan distances in an
euclidian space of 2 dimensions. We use classic <strong>multi dimensional scaling</strong> to produce 2D-coordinates
from manhattan distance values.</p>
<ul>
<li>Given distances ${d_{ij}}$ between genotypes ${i}$ and ${j}$</li>
<li>Find positions ${(x_{ij}, y_{ij})}$</li>
<li>With maximum dimension of the space which the data are to be represented in ${k = 2}$</li>
<li>That minimize a stress function</li>
</ul>
<p>$$
\sum_{k=1}^{k=2}\sum_{i}\sum_{j} (|x_{ik}-x_{jk}| - d_{ij})^2
$$</p>
<p>In base R, MDS is available as <code>cmdscale</code> function.</p>
<pre><code>fit &lt;- cmdscale(vcfDist, eig = TRUE, k = 2)

fitVcfDistDf &lt;- data.frame(
  genotype = row.names(fit$points),
  x = fit$points[, 1],
  y = fit$points[, 2]
)

ggplot(fitVcfDistDf, aes(x = x, y = y, label = genotype)) +
  geom_point() +
  geom_text(hjust = 0, vjust = 0) +
  theme_minimal() +
  xlim(-4000, 16000) +
  xlab(&quot;&quot;) +
  ylab(&quot;&quot;)
</code></pre>
<p><img src="https://guerinpe.com/articles/multi-dimensional-scaling/./example_plot_multidimensional_scaling.jpg" alt="example plot multidimensional scaling" /></p>
<p>The MDS reveals that the genotypes <code>J</code> and <code>F</code> are distinct from the rest of the samples.</p>
<h2 id="conclusion">Conclusion</h2>
<ul>
<li>PCA creates plots based on correlations among samples. It is a visualization of variance.</li>
<li>MDS create plots based on distances or dissimilarity among samples. It is a visualization of similarity.</li>
</ul>
<h2 id="references">References</h2>
<blockquote>
<p><strong>Multidimensional scaling for large genomic data sets</strong></p>
<p><em>Jengnan Tzeng, Henry Horng-Shing Lu and Wen-Hsiung Li</em></p>
<p>BMC Bioinformatics, 2008. DOI: <a href="https://doi.org/10.1186/1471-2105-9-179">10.1186/1471-2105-9-179</a></p>
</blockquote>
<blockquote>
<p><strong>Multidimensional scaling: I. theory and method.</strong></p>
<p><em>Warren S Torgerson</em></p>
<p>Psychometrika, 1952. DOI: <a href="https://doi.org/10.1007/BF02288916">10.1007/BF02288916</a></p>
</blockquote>

  <br>
  <br>
  <br>
  <br>
  <hr>
  
<div class="post-content-wrapper">
    <div class="post-info">
    Published on April 3, 2022
    <br><br>
    </div>
    
    <hr class="bold">     
    <div class="home-latest-articles">
        <h2 class="home-section-title">Relevant Tags</h2>
        <div class="tags-article">
            <ul>
                 
                
                <li style="background: #98D7E4">
                <a style="color: #0D3B44" 
                   href="https://guerinpe.com/tags/data-science/">data-science</a>&nbsp;
                </li> 
                
                        
            </ul>    
        </div>
    </div>

    <div class="home-latest-articles">
        <h2 class="home-section-title">About the Author</h2>

        <div class="about-author">
            <div class="author-container">
                <img src="https:&#x2F;&#x2F;guerinpe.com/avatar.png" alt="Portrait of Pierre-Edouard Guerin Bioinformatician Data Scientist"/>
                <div class="author-info">
                    <a href="https:&#x2F;&#x2F;guerinpe.com" class="author-name">Pierre-Edouard Guerin</a>
                    <div class="author-description">Bioinformatician</div>
                    <div class="brand_nav">
                        
                        
                        <a href="https:&#x2F;&#x2F;www.linkedin.com&#x2F;in&#x2F;pierre-edouard-guerin"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M20.447 20.452h-3.554v-5.569c0-1.328-.027-3.037-1.852-3.037-1.853 0-2.136 1.445-2.136 2.939v5.667H9.351V9h3.414v1.561h.046c.477-.9 1.637-1.85 3.37-1.85 3.601 0 4.267 2.37 4.267 5.455v6.286zM5.337 7.433c-1.144 0-2.063-.926-2.063-2.065 0-1.138.92-2.063 2.063-2.063 1.14 0 2.064.925 2.064 2.063 0 1.139-.925 2.065-2.064 2.065zm1.782 13.019H3.555V9h3.564v11.452zM22.225 0H1.771C.792 0 0 .774 0 1.729v20.542C0 23.227.792 24 1.771 24h20.451C23.2 24 24 23.227 24 22.271V1.729C24 .774 23.2 0 22.222 0h.003z"/></svg> </a>
                        
                        <a href="https:&#x2F;&#x2F;twitter.com&#x2F;guerin_pe"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.954 4.569c-.885.389-1.83.654-2.825.775 1.014-.611 1.794-1.574 2.163-2.723-.951.555-2.005.959-3.127 1.184-.896-.959-2.173-1.559-3.591-1.559-2.717 0-4.92 2.203-4.92 4.917 0 .39.045.765.127 1.124C7.691 8.094 4.066 6.13 1.64 3.161c-.427.722-.666 1.561-.666 2.475 0 1.71.87 3.213 2.188 4.096-.807-.026-1.566-.248-2.228-.616v.061c0 2.385 1.693 4.374 3.946 4.827-.413.111-.849.171-1.296.171-.314 0-.615-.03-.916-.086.631 1.953 2.445 3.377 4.604 3.417-1.68 1.319-3.809 2.105-6.102 2.105-.39 0-.779-.023-1.17-.067 2.189 1.394 4.768 2.209 7.557 2.209 9.054 0 13.999-7.496 13.999-13.986 0-.209 0-.42-.015-.63.961-.689 1.8-1.56 2.46-2.548l-.047-.02z"/></svg> </a>
                        
                        <a href="https:&#x2F;&#x2F;github.com&#x2F;Grelot"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/></svg> </a>
                        
                        <a href="https:&#x2F;&#x2F;gitlab.mbb.univ-montp2.fr&#x2F;users&#x2F;peguerin&#x2F;groups"> <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.955 13.587l-1.342-4.135-2.664-8.189c-.135-.423-.73-.423-.867 0L16.418 9.45H7.582L4.919 1.263C4.783.84 4.185.84 4.05 1.26L1.386 9.449.044 13.587c-.121.375.014.789.331 1.023L12 23.054l11.625-8.443c.318-.235.453-.647.33-1.024"/></svg> </a>
                        
                        
                    </div>
                </div>
            </div>
        </div>
    </div>

    


<div class="home-latest-articles">
  <h2 class="home-section-title">Latest Articles</h2>

 <ul class="home-article-list">
  
    <li class="home-article-item">
      <a href="https:&#x2F;&#x2F;guerinpe.com&#x2F;articles&#x2F;research-tax-credit&#x2F;" class="home-article-link">
        <h3 class="home-article-title">Research Tax Credit</h3>
        <div class="home-article-summary">The Research Activities Tax Credit is a tax credit that incentivizes private companies to increase their Research and Development (R&amp;D).  Within my company, I have been tasked with writing the Research Tax Credit (CIR) justification report for France. Here, the method for writing such a report.</div>
        <div class="home-article-date">  OCT 2025 · PIERRE-EDOUARD GUERIN </div>
      </a>
      
    </li>
  
    <li class="home-article-item">
      <a href="https:&#x2F;&#x2F;guerinpe.com&#x2F;articles&#x2F;turing-complete-from-logical-gates-to-cpu-architecture&#x2F;" class="home-article-link">
        <h3 class="home-article-title">Turing Complete: From Logical Gates to CPU Architecture</h3>
        <div class="home-article-summary">In 2021, LevelHead published Turing Complete, a game about computer science. My friend Christophe Georgescu recommended me to play it. Unfortunately, I took his advice and now I can not stop to play this game! The game challenges you to design an entire computer from scratch. You start with basic logic gates, then move on to components, memory, CPU architecture, and finally assembly programming. By the way, the game is neat and present all these concepts in a playful and intuitive way.</div>
        <div class="home-article-date">  SEP 2025 · PIERRE-EDOUARD GUERIN </div>
      </a>
      
    </li>
  
    <li class="home-article-item">
      <a href="https:&#x2F;&#x2F;guerinpe.com&#x2F;articles&#x2F;how-to-manage-project&#x2F;" class="home-article-link">
        <h3 class="home-article-title">How to Manage a Project?</h3>
        <div class="home-article-summary">In any company, every task is part of a project. I am responsible for managing multiple projects each year. I have to present deliverables to stakeholders, meet deadlines, allocate mandays and coordinate everyone’s actions. This is a meticulous work that requires a strong methodology.</div>
        <div class="home-article-date">  JUN 2025 · PIERRE-EDOUARD GUERIN </div>
      </a>
      
    </li>
  
</ul>


  <div class="home-see-all-container">
    <button class="home-see-all-button ">
    <a href="https:&#x2F;&#x2F;guerinpe.com/articles" role="button">See all &gt;</a>
    </button>
  </div>
</div>
</div>

</div>


    <br>

  <footer class="footer">
  <h2>Site Plan</h2>
  <div class="footer-inner">
    <div class="footer-links">
      <a href="https://guerinpe.com/.">Home</a>
      <a href="https://guerinpe.com/articles">Blog</a>
      <a href="https://guerinpe.com/resume/">CV</a>
      <a href="https://guerinpe.com/activity">Activity</a>
      <a href="https://guerinpe.com/publications">Publications</a>
      <a href="https://guerinpe.com/search">Search</a>
    </div>
</div>

  <h2>Tags</h2>


  
  <mark>Biotech</mark>
  
  <mark>Career</mark>
  
  <mark>Communication</mark>
  
  <mark>Computer-science</mark>
  
  <mark>Data-engineering</mark>
  
  <mark>Data-science</mark>
  
  <mark>Management</mark>
  
  <mark>Math</mark>
  
  <mark>Med-science</mark>
  
  <mark>Omics</mark>
  
  <mark>Plant-science</mark>
  
  <mark>Sciences</mark>
  
  <mark>Society</mark>
  
  <mark>Software-engineering</mark>
  
  <mark>Sys</mark>
  
  <mark>Tech</mark>
  
  <mark>Zoo</mark>
    


  <div class="footer-inner">

    <h2>Social Networks</h2>

    <div class="footer-social">
        
        
        <a href="https:&#x2F;&#x2F;www.linkedin.com&#x2F;in&#x2F;pierre-edouard-guerin" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M20.447 20.452h-3.554v-5.569c0-1.328-.027-3.037-1.852-3.037-1.853 0-2.136 1.445-2.136 2.939v5.667H9.351V9h3.414v1.561h.046c.477-.9 1.637-1.85 3.37-1.85 3.601 0 4.267 2.37 4.267 5.455v6.286zM5.337 7.433c-1.144 0-2.063-.926-2.063-2.065 0-1.138.92-2.063 2.063-2.063 1.14 0 2.064.925 2.064 2.063 0 1.139-.925 2.065-2.064 2.065zm1.782 13.019H3.555V9h3.564v11.452zM22.225 0H1.771C.792 0 0 .774 0 1.729v20.542C0 23.227.792 24 1.771 24h20.451C23.2 24 24 23.227 24 22.271V1.729C24 .774 23.2 0 22.222 0h.003z"/></svg>
        </a>
      
        <a href="https:&#x2F;&#x2F;twitter.com&#x2F;guerin_pe" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.954 4.569c-.885.389-1.83.654-2.825.775 1.014-.611 1.794-1.574 2.163-2.723-.951.555-2.005.959-3.127 1.184-.896-.959-2.173-1.559-3.591-1.559-2.717 0-4.92 2.203-4.92 4.917 0 .39.045.765.127 1.124C7.691 8.094 4.066 6.13 1.64 3.161c-.427.722-.666 1.561-.666 2.475 0 1.71.87 3.213 2.188 4.096-.807-.026-1.566-.248-2.228-.616v.061c0 2.385 1.693 4.374 3.946 4.827-.413.111-.849.171-1.296.171-.314 0-.615-.03-.916-.086.631 1.953 2.445 3.377 4.604 3.417-1.68 1.319-3.809 2.105-6.102 2.105-.39 0-.779-.023-1.17-.067 2.189 1.394 4.768 2.209 7.557 2.209 9.054 0 13.999-7.496 13.999-13.986 0-.209 0-.42-.015-.63.961-.689 1.8-1.56 2.46-2.548l-.047-.02z"/></svg>
        </a>
      
        <a href="https:&#x2F;&#x2F;github.com&#x2F;Grelot" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12"/></svg>
        </a>
      
        <a href="https:&#x2F;&#x2F;gitlab.mbb.univ-montp2.fr&#x2F;users&#x2F;peguerin&#x2F;groups" target="_blank" rel="noopener noreferrer">
          <svg class="brand-icon" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path fill="currentColor" d="M23.955 13.587l-1.342-4.135-2.664-8.189c-.135-.423-.73-.423-.867 0L16.418 9.45H7.582L4.919 1.263C4.783.84 4.185.84 4.05 1.26L1.386 9.449.044 13.587c-.121.375.014.789.331 1.023L12 23.054l11.625-8.443c.318-.235.453-.647.33-1.024"/></svg>
        </a>
      
      
    </div>
  </div>
</div>
  
  


  <h2>Site Information</h2>

  <div class="footer-meta">
    <p>
     2015-2025
     guerinpe.com by Pierre-Edouard GUERIN is under a 
      <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a> ·
      <a href="https://www.getzola.org/">Powered by Zola</a> ·
      <a href="https://guerinpe.com/atom.xml">RSS feed</a>
    </p>
  </div>
</footer>

  </body>



</html>
