+++
template = "page_static.html"
title = "Curriculum Vitae"
date =  2025-06-25
draft = false
tags = ["resume"]
+++



* [E-mail contact](mailto:pierre-edouard.guerin@groupefd.com)
* [Linkedin profile](https://www.linkedin.com/in/pierre-edouard-guerin/)
* [ORCiD profile](https://orcid.org/0000-0001-7909-3729)

_______________________________________________________________________________

## Experience
<div class="timeline-entry">
  <div class="timeline-date">July 2021 to present</div>
  <div class="timeline-item">
    <h3>Bioinformatician</h3>
    <h5>Florimond Desprez Group Research Unit</h5>
    <h6>Cappelle-en-Pévèle, France</h6>
    <p>
      Developed and optimised software, workflows, and databases to support plant breeders in the analysis and interpretation of omics data.
    </p>
      <mark>Python</mark>
      <mark>SQL</mark>
      <mark>R</mark>
      <mark>C++</mark>
      <mark>Docker</mark>
      <mark>Nextflow</mark>
      <mark>Azure</mark>
      <mark>Databricks</mark>
      <mark>Elastic Stack</mark>
  </div>

  <div class="timeline-date">January 2017 to July 2021</div>
  <div class="timeline-item">
    <h3>Bioinformatician</h3>
    <h5>CNRS Research Unit 5175, Center of Functional Ecology and Evolution</h5>
    <h6>Montpellier, France</h6>
    <p>
      Processed DNA sequence data for population genetics studies of various fish species, enabling marine biologists to assess their conservation status.
    </p>
      <mark>Python</mark>
      <mark>R</mark>
      <mark>C++</mark>
      <mark>Singularity</mark>
      <mark>Snakemake</mark>
      <mark>Jupyter</mark>
      <mark>SGE</mark>
      <mark>SLURM</mark>
  </div>

  <div class="timeline-date">January to August 2016</div>
  <div class="timeline-item">
    <h3>Bioinformatics Internship</h3>
    <h5>INSERM Unit S598, Genetics of Diabetes</h5>
    <h6>Paris, France</h6>
    <p>
      Developed a module and user interface for a medical software to detect and annotate rare variants in human genome sequences related to diabetes.
    </p>
      <mark>SQL</mark>
      <mark>Python</mark>
      <mark>Qt</mark>
  </div>

  <div class="timeline-date">January to June 2015</div>
  <div class="timeline-item">
    <h3>Bioinformatics Internship</h3>
    <h5>INSERM Unit S1134, Macromolecular Biology</h5>
    <h6>Paris, France</h6>
    <p>
      Optimised an algorithm to predict protein 3D structure at atomic resolution.
    </p>
      <mark>C</mark>
      <mark>Python</mark>
  </div>
</div>


	
## Education

* **2016: M.Sc. in Bioinformatics**, [Paris University](https://u-paris.fr/), France
* **2014: Licence in Bioinformatics**, [Paris University](https://u-paris.fr/), France

## Training

* **2024: Elastic Stack Workshop**, [AMBIENT-IT](https://www.ambient-it.net/), France
* **2023: Databricks Lakehouse Bootcamp**, [Databricks Academy](https://www.databricks.com/learn/training/home), Belgium
* **2021: Landscape Genetics Distributed Graduate Seminar**, [IDGC](https://sites.google.com/site/landscapegeneticscourse/), USA
* **2019: Nextflow Training**, [CRG](https://www.crg.eu/en/event/coursescrg-nextflow-2019), Catalunya
* **2018: Advanced Statistics for Data Sciences**, [EPHE](https://www.ephe.psl.eu/formations-conferences), France
* **2017: High-Performance Computing Workshop**, [MBB](https://isem-evolution.fr/plateau/plateau-montpellier-bioinformatique-et-biodiversite/), France


_______________________________________________________________

<br>
<br>

# Projects

## Florimond Desprez Group

My projects for the company are **confidentials**. Please contact me for more information.

## Public research

<div class="all_proj">


<div role="button" class="project" onclick="location.href='https://github.com/Grelot/genbar2'">
<img align="center" width="48rem" height="48rem" src="cnrs.png">
<div class="title"> Genbar 2 </div>
<div class="description"> I programmed a software to identify genetic boundaries between populations from individual spatial coordinates and genetic variants data.</div>
<div class="skills"> (C, C++, htslib)</div>
<hr width="31%"> 
<div class="duration"> May 2019 - Feb 2020 </div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/edna'">
<img align="center" width="128rem" height="48rem" src="monaco.png">
<div class="title">Metabarcoding</div>
<div class="description">I programmed several workflow to process metabarcoding environmental DNA data from MONACO MARINE WORLDWIDE EXPEDITION.</div>
<div class="skills"> (obitools, vsearch, swarm, cutadapt, bash, python, singularity, snakemake)</div>
<hr width="31%"> 
<div class="duration"> Jan 2019 - Jul 2021</div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/lmathon/eDNA--benchmark_pipelines'">
<img align="center" width="114rem" height="48rem" src="spygen.png">
<div class="title">Benchmarking of metabarcoding workflows</div>
<div class="description">We tested some combinations of softwares to improve performances of metabarcoding data processing.</div>
<div class="skills"> (obitools, vsearch, qiime, python, singularity, Univa Grid Engine)</div>
<hr width="31%"> 
<div class="duration"> Mar 2018 - Jul 2021</div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/reservebenefit/snakemake_stacks2'">
<img align="center" width="48rem" height="48rem" src="reservebenefit.png">
<div class="title">Population genomics</div>
<div class="description">I genotyped 1200 individuals belonging to 3 fish species. I worked with restriction enzyme-based data such as RAD-seq.</div>
<div class="skills"> (illumina paired-end, STACKS, vcftools, bedtools, bwa, python, snakemake, singularity, Univa Grid Engine, bash)</div>
<hr width="31%"> 
<div class="duration"> Jun 2017 - Oct 2019 </div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/Grelot/aker--beetGenomeEnvironmentAssociation'">
<img align="center" width="48rem" height="48rem" src="florimond.png">
<div class="title">Beets genome metrics</div>
<div class="description">I calculated metrics (nucleotide diversity, Tajima's D) on the beets genome from 14,409 random single nucleotide polymorphisms (SNPs) among 299 accessions of cultivated beets.</div>
<div class="skills"> (R, python, genpop)</div>
<hr width="31%"> 
<div class="duration"> May 2017 - May 2018 </div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/Grelot/global_fish_genetic_diversity'">
<img align="center" width="224rem" height="48rem" src="ephe.png">
<div class="title">First Global Map of Fish Genetic Diversity</div>
<div class="description"> I built a database containing over 50,000 DNA sequences representing 3,815 species of marine fish and 1,611 species of freshwater fish. I estimated the average genetic diversity at different geographical scales.</div>
<div class="skills"> (julia, python, R, singularity, MUSCLE, UGENE, geonames, BOLD, shiny)</div>
<hr width="31%"> 
<div class="duration"> May 2017 - Feb 2020 </div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/seaconnect'">
<img align="center" width="136rem" height="48rem" src="total.png">
<div class="title">Landscape genomics</div>
<div class="description">I processed low-coverage RAD-seq data from 1800 individuals belonging to 2 fish species collected from all over the Mediterranean sea.</div>
<div class="skills"> (dDocent, freebayes, vcftools, samtools, trimmomatic, bash, python, singularity, snakemake)</div>
<hr width="31%"> 
<div class="duration"> Feb 2017 - Mar 2021 </div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/reservebenefit/genomic_resources_for_med_fishes'">
<img align="center" width="48rem" height="48rem" src="reservebenefit.png">
<div class="title">Genome assembly</div>
<div class="description">I did the sequencing and assembly of 3 new fish species nuclear and mitochondrial genomes.</div>
<div class="skills"> (illumina paired-end, mate-pair, 10X genomics chromium, Abyss, Platanus, QUAST, SLURM, bash)</div>
<hr width="31%"> 
<div class="duration"> Jan 2017 - Nov 2019 </div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/Grelot/diabetesGenetics--COAT'">
<img align="center" width="136rem" height="32rem" src="inserm.png">
<div class="title">Report bad quality region of coding sequences from genome sequencing data</div>
<div class="description">I developped a software able to detect human genomic variations with low coverage. Graphical Interface. </div>
<div class="skills"> (Illumina paired-end, samtools, bedtools, variation annotation, python, mySQL, qt4)</div>
<hr width="31%"> 
<div class="duration"> Jan 2016 - Jul 2016 </div>
</div>

<div role="button" class="project" onclick="location.href='https://www.dsimb.inserm.fr/orion/'">
<img align="center" width="136rem" height="32rem" src="inserm.png">
<div class="title">Optimization of a method of fold recognition in protein structure</div>
<div class="description">I added a new algorithm to predict 3D-modelisation of protein structure at atomic resolution </div>
<div class="skills"> (PDB, pymol, C, C++, python, R, html, css )</div>
<hr width="31%"> 
<div class="duration"> Feb 2015 - Jun 2015 </div>
</div>

</div>


_______________________________________________________________________________
 

# Other activities

* My scientific [blog](https://guerinpe.com/articles/)
* Member of the association of Young French Bioinformaticians [Jebif](https://jebif.fr/en/)
* Author on french bioinformatics participative blog [bioinfo-fr](https://bioinfo-fr.net/author/pierre-edouard-guerin)
* My [data science skills demo](https://github.com/Grelot/machine_learning_demo) *(linear regression, random forest, neural network, bayesian, deep learning)* using python ecosystem for machine learning *(pandas, numpy, tensorflow, scikit-learn)* training on [kaggle](https://www.kaggle.com/pierreedouardguerin).







