+++
template = "static_page.html"
title = "Curriculum Vitae"
date =  2019-08-20
draft = false
tags = ["education", "profile"]
+++

I am a computational biologist in [CEFE](https://www.cefe.cnrs.fr) research team at [CNRS](http://www.cnrs.fr/languedoc-roussillon/03com-labos/03-2-lab-site/02-lab-site-montp.htm) of Montpellier, France. 


## Areas of focus

* **Bioinformatics**
	* Metabarcoding
	* Genomics
* **Linux Systems**
	* Ubuntu and Debian based systems. Orchestration and automation of tasks ranging from containerized web services to DNA high throughput sequencing data processing and workflow management on HPC platform.
* **Programming Languages**
	* Bash, Python, C, C++, R, julia

_______________________________________________________________________________

# EXPERIENCE

## [Center of Functional Ecology and Evolution](https://www.cefe.cnrs.fr/)

<a href="https://www.ephe.fr/"><img align="right" width="132rem" height="124rem" src="ephe.png"></a>
<a href="https://www.cnrs.fr/"><img align="right" width="96rem" height="96rem" src="cnrs.png"></a>


> **Computational biologist**
> <br> Montpellier, FR
> <br> *02.2017 - current*

- Landscape genomics
- Metabarcoding
- Genome assembly


## [Genetics of Diabetes](https://ecolesdoctorales.parisdescartes.fr/ed157/Equipes-de-recherche/Liste-des-equipes-d-accueil/Genetique-des-diabetes)

<a href="https://www.inserm.fr/"><img align="right" width="164rem" height="36rem" src="inserm.png"></a>

> **Software developer** (Master internship)
> <br> Paris, FR
> <br> *01.2016 - 08.2016*

- Human genomes DNA high throughput sequencing data analysis
- Program to detect and annotate rare genetic variants
- Graphical interface

## [Dynamics of Structures and Interactions of Macromolecules in Biology](http://www.dsimb.inserm.fr/)

<a href="https://www.inserm.fr/"><img align="right" width="164rem" height="36rem" src="inserm.png"></a>


> **Computational biologist** (Master internship)
> <br> Paris, FR
> <br> *02.2015 - 06.2015*

- 3D-modelisation of protein structure at atomic resolution
- Optimization of a method of fold recognition in protein structure
- New algorithm to predict protein structure

# PROJECTS

blablalbla 
blablalbla 
blablalbla 

blablalbla 
blablalbla 
blablalbla 
blablalbla 
* test
* test
* test
* test
* test


# EDUCATION

<img align="right" width="96rem" height="192rem" src="paris7.png">


## University of Paris 
> **MASTER Degree of Bioinformatics**
> <br> Paris, FR
> <br> *june 2016*



# SKILLS

## Soft Skills

* Interacted with many researchers from diverse fields for co-authored research work
* Technical support and training for students and partners
* Presentations of my works at several conferences, at group and institute-wide meeting
* Technical consulting and participation in decision-making

## Software Engineering

| Domain                    | Tools                      |
|---------------------------|----------------------------|
| Operating systems       | Ubuntu, Debian             |
| Programming languages     | Python, R, C++, bash, C, julia   |
| Graphical interface       | Qt                        
| Database                  | PostgresSQL, mySQL         |
| Web technologies          | HTML, CSS, SVG, javascript, django |
| Workload manager          | SLURM, Univa Grid Engine |
| Version control system    | Git, gitlab, github       |
| Manage a container        | Singularity, docker        |
| Workflow manager          | Snakemake, nextflow       |

## Statistics

* Statistical modeling
* Machine learning and optimization theory
* Bayesian inference

## Bioinformatics

| Domain                    | Softwares                  |
|---------------------------|----------------------------|
| Metabarcoding             | Vsearch, usearch, swarm, obitools2 |
| Population genomics       | STACKS, freebayes, dDocent, STRUCTURE |
| Genomics                  | VCFtools, SAMtools, bedtools, cutadapt, trimmomatic, bwa, annovar... |

## Design

* Inkscape
* Adobe photoshop
* Adobe illustrator


# SOFTWARES

* [Genbar2](https://github.com/Grelot/genbar2): identify genetic boundaries between populations using individual spatial coordinates and genetic variants
* [DEMORT](https://pypi.org/project/demort/): a DEmultiplexing MOnitoring Report Tool
* [EXAM](https://sourceforge.net/projects/exam-exome-analysis-and-mining/): a whole exome sequencing analysis and its graphical interface
* [COAT](https://github.com/Grelot/diabetesGenetics--COAT): a COverage Analysis Tool for exome sequencing data
* [ORION](http://www.dsimb.inserm.fr/ORION/): a sensivitve method for protein template detection


# PUBLICATIONS


* **Global determinants of freshwater and marine fish genetic diversity** (submitted to Nature Communications in februrary 2019)
* **Predicting genotype environmental range from genomeâ€“environment associations** ([Molecular Ecology in may 2018](https://doi.org/10.1111/mec.14723))
* **ORION : a web server for protein fold recognition and structure prediction using evolutionary hybrid profiles** ([Scientific Reports in june 1016](https://doi.org/10.1038/srep28268))





_______________________________________________________________________________

# OTHER ACTIVITIES


## Sciences

* Member of the association of Young French Bioinformaticians [Jebif](https://jebif.fr/en/)
* Author on french bioinformatics participative blog [bioinfo-fr](https://bioinfo-fr.net/author/pierre-edouard-guerin)


## Personnal projects

* [speckyman](https://github.com/Grelot/speckyman): a platform game developed in JavaScript
* [fromdnatomusic](https://github.com/Grelot/fromdnatomusic): convert DNA sequences into MIDI track and listen to it




## Triathlon

<img align="right" width="276rem" height="256rem" src="usnm.png">

I'm an active member of the [USN-MONTPELLIER](https://www.usn-montpellier.fr/usn-web/view/index.php) sportive club since 2017. Join us !