+++
template = "page_static.html"
title = "Curriculum Vitae"
date =  2025-06-25
draft = false
+++



* [E-mail contact](mailto:pierre-edouard.guerin@groupefd.com)
* [Linkedin profile](https://www.linkedin.com/in/pierre-edouard-guerin/)
* [Kaggle portfolio](https://www.kaggle.com/pierreedouardguerin)


## Summary

I am a bioinformatician who designs tools to help breeders improve the genetics of industrial crops. My tools process, store and integrate [omics data](/activity/) into databases linking genetic information with phenotypes and field conditions. I am interested in any technology related to the processing, visualization or mining of biodata.

## Experience
<div class="timeline-entry">
  <div class="timeline-date">July 2021 to present</div>
  <div class="timeline-item">
    <h3>Bioinformatician</h3>
    <h5>Florimond Desprez Group Research Unit</h5>
    <h6>Cappelle-en-Pévèle, France</h6>
    <p>
      Developed and optimised software, workflows, and databases to support plant breeders in the analysis and interpretation of omics data.
    </p>
      <mark>Python</mark>
      <mark>SQL</mark>
      <mark>R</mark>
      <mark>C++</mark>
      <mark>Docker</mark>
      <mark>Nextflow</mark>
      <mark>Azure</mark>
      <mark>Databricks</mark>
      <mark>Elastic Stack</mark>
  </div>

  <div class="timeline-date">January 2017 to July 2021</div>
  <div class="timeline-item">
    <h3>Bioinformatician</h3>
    <h5>CNRS Research Unit 5175, Center of Functional Ecology and Evolution</h5>
    <h6>Montpellier, France</h6>
    <p>
      Processed DNA sequence data for population genetics studies of various fish species, enabling marine biologists to assess their conservation status.
    </p>
      <mark>Python</mark>
      <mark>R</mark>
      <mark>C++</mark>
      <mark>Singularity</mark>
      <mark>Snakemake</mark>
      <mark>Jupyter</mark>
      <mark>SGE</mark>
      <mark>SLURM</mark>
  </div>

  <div class="timeline-date">January to August 2016</div>
  <div class="timeline-item">
    <h3>Bioinformatics Internship</h3>
    <h5>INSERM Unit S598, Genetics of Diabetes</h5>
    <h6>Paris, France</h6>
    <p>
      Developed a module and user interface for a medical software to detect and annotate rare variants in human genome sequences related to diabetes.
    </p>
      <mark>SQL</mark>
      <mark>Python</mark>
      <mark>Qt</mark>
  </div>

  <div class="timeline-date">January to June 2015</div>
  <div class="timeline-item">
    <h3>Bioinformatics Internship</h3>
    <h5>INSERM Unit S1134, Macromolecular Biology</h5>
    <h6>Paris, France</h6>
    <p>
      Optimised an algorithm to predict protein 3D structure at atomic resolution.
    </p>
      <mark>C</mark>
      <mark>Python</mark>
  </div>
</div>

## Training

* 2024: Generative AI workshop, [Interactivity](https://www.interactivity.nl/)
* 2024: Elastic Stack Workshop, [AMBIENT-IT](https://www.ambient-it.net/)
* 2023: Databricks Lakehouse Bootcamp, [Databricks Academy](https://www.databricks.com/learn/training/home)
* 2020: Landscape Genetics Distributed Graduate Seminar, [IDGC](https://sites.google.com/site/landscapegeneticscourse/)
* 2019: ReproHackathon, [CIRAD](https://github.com/IFB-ElixirFr/ReproHackathon)
* 2018: Advanced Statistics for Data Sciences, [EPHE](https://www.ephe.psl.eu/formations-conferences)
* 2017: High-Performance Computing Workshop, [MBB](https://isem-evolution.fr/plateau/plateau-montpellier-bioinformatique-et-biodiversite/)



	
## Education

* **2016: M.Sc. in Bioinformatics**, [Paris University](https://u-paris.fr/), France
* **2014: Licence in Bioinformatics**, [Paris University](https://u-paris.fr/), France

