+++
template = "page.html"
title = "Curriculum Vitae"
date =  2025-06-25
draft = false
tags = ["resume"]
+++

## Bioinformatician at FLORIMOND DESPREZ GROUP
<a href="mailto:pierre.edouard.guerin@gmail.com"> E-mail: pierre.edouard.guerin [at] gmail.com</a>


## Experience


<div class="timeline-item" date-is='Jul 2021 to present'>
    <h3>Bioinformatician</h3>
    <h6>FLORIMOND DESPREZ GROUP Research Unit, Cappelle-en-Pevele</h6> 
    <p>            
        I bring my skills and support to the field crops selection department by improving or designing methods to process
        omics data.
    </p> 
        <mark>python</mark>
        <mark>SQL</mark>            
        <mark>R</mark>
        <mark>C++</mark>
        <mark>Docker</mark>
        <mark>Nextflow</mark>
        <mark>Azure cloud</mark>
        <mark>Databricks</mark>
        <mark>Elastic stack</mark>
</div>

<div class="timeline-item" date-is='Feb 2017 to Jul 2021'>
    <h3>Bioinformatician</h3>
    <h6>CNRS Research Unit 5175, Center of Functional Ecology and Evolution, Montpellier</h6> 
    <p>            
        I develop softwares for the analysis and visualization of data from high-throughput DNA sequencing (genomics, metagenomics, environmental DNA). I also ensure technology watch to implement new data processing methods and optimize scientific reproducibility.
    </p> 
        <mark>python</mark>            
        <mark>R</mark>
        <mark>C++</mark>
        <mark>Singularity</mark>
        <mark>Snakemake</mark>
        <mark>Jupyter notebook</mark>
        <mark>SGE</mark>
        <mark>SLURM</mark>
</div>	
<div class="timeline-item" date-is='Jan to Aug 2016'>
    <h3>Bioinformatics internship</h3>
    <h6>INSERM Research Unit S598, Genetics of Diabetes, Paris</h6>
    <p>
        I developed and integrated a functionnality and its graphical interface to a medical software. It aims to detect and annotate rare genetic variants related to diabetes on human genome sequence data.
    </p>
        <mark>SQL</mark>
        <mark>python</mark>            
        <mark>Qt</mark>
</div>	
<div class="timeline-item" date-is='Feb to June 2015'>
    <h3>Bioinformatics internship</h3>
    <h6>INSERM Research Unit S1134, Dynamics of Structures and Interactions of Macromolecules in Biology, Paris</h6>
    <p>
        Improvement and implementation of an algorithm to predict 3D-modelisation of protein structure at atomic resolution.
    </p>
        <mark>C</mark>            
        <mark>python</mark>
</div>
	
## Education

* **2016: M.Sc. in Bioinformatics**, Université Paris Diderot, France
* **2014: Licence in Bioinformatics**, Université Paris Diderot, France

_______________________________________________________________________________

# Projects

## FLORIMOND DESPREZ Group

My projects for the company are **confidentials**. Please contact me for more information.

## Public research

<div class="all_proj">


<div role="button" class="project" onclick="location.href='https://github.com/Grelot/genbar2'">
<img align="center" width="48rem" height="48rem" src="cnrs.png">
<div class="title"> Genbar 2 </div>
<div class="description"> I programmed a software to identify genetic boundaries between populations from individual spatial coordinates and genetic variants data.</div>
<div class="skills"> (C, C++, htslib)</div>
<hr width="31%"> 
<div class="duration"> May 2019 - Feb 2020 </div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/edna'">
<img align="center" width="128rem" height="48rem" src="monaco.png">
<div class="title">Metabarcoding</div>
<div class="description">I programmed several workflow to process metabarcoding environmental DNA data from MONACO MARINE WORLDWIDE EXPEDITION.</div>
<div class="skills"> (obitools, vsearch, swarm, cutadapt, bash, python, singularity, snakemake)</div>
<hr width="31%"> 
<div class="duration"> Jan 2019 - Jul 2021</div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/lmathon/eDNA--benchmark_pipelines'">
<img align="center" width="114rem" height="48rem" src="spygen.png">
<div class="title">Benchmarking of metabarcoding workflows</div>
<div class="description">We tested some combinations of softwares to improve performances of metabarcoding data processing.</div>
<div class="skills"> (obitools, vsearch, qiime, python, singularity, Univa Grid Engine)</div>
<hr width="31%"> 
<div class="duration"> Mar 2018 - Jul 2021</div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/reservebenefit/snakemake_stacks2'">
<img align="center" width="48rem" height="48rem" src="reservebenefit.png">
<div class="title">Population genomics</div>
<div class="description">I genotyped 1200 individuals belonging to 3 fish species. I worked with restriction enzyme-based data such as RAD-seq.</div>
<div class="skills"> (illumina paired-end, STACKS, vcftools, bedtools, bwa, python, snakemake, singularity, Univa Grid Engine, bash)</div>
<hr width="31%"> 
<div class="duration"> Jun 2017 - Oct 2019 </div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/Grelot/aker--beetGenomeEnvironmentAssociation'">
<img align="center" width="48rem" height="48rem" src="florimond.png">
<div class="title">Beets genome metrics</div>
<div class="description">I calculated metrics (nucleotide diversity, Tajima's D) on the beets genome from 14,409 random single nucleotide polymorphisms (SNPs) among 299 accessions of cultivated beets.</div>
<div class="skills"> (R, python, genpop)</div>
<hr width="31%"> 
<div class="duration"> May 2017 - May 2018 </div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/Grelot/global_fish_genetic_diversity'">
<img align="center" width="224rem" height="48rem" src="ephe.png">
<div class="title">First Global Map of Fish Genetic Diversity</div>
<div class="description"> I built a database containing over 50,000 DNA sequences representing 3,815 species of marine fish and 1,611 species of freshwater fish. I estimated the average genetic diversity at different geographical scales.</div>
<div class="skills"> (julia, python, R, singularity, MUSCLE, UGENE, geonames, BOLD, shiny)</div>
<hr width="31%"> 
<div class="duration"> May 2017 - Feb 2020 </div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/seaconnect'">
<img align="center" width="136rem" height="48rem" src="total.png">
<div class="title">Landscape genomics</div>
<div class="description">I processed low-coverage RAD-seq data from 1800 individuals belonging to 2 fish species collected from all over the Mediterranean sea.</div>
<div class="skills"> (dDocent, freebayes, vcftools, samtools, trimmomatic, bash, python, singularity, snakemake)</div>
<hr width="31%"> 
<div class="duration"> Feb 2017 - Mar 2021 </div>
</div>

<div role="button" class="project" onclick="location.href='https://gitlab.mbb.univ-montp2.fr/reservebenefit/genomic_resources_for_med_fishes'">
<img align="center" width="48rem" height="48rem" src="reservebenefit.png">
<div class="title">Genome assembly</div>
<div class="description">I did the sequencing and assembly of 3 new fish species nuclear and mitochondrial genomes.</div>
<div class="skills"> (illumina paired-end, mate-pair, 10X genomics chromium, Abyss, Platanus, QUAST, SLURM, bash)</div>
<hr width="31%"> 
<div class="duration"> Jan 2017 - Nov 2019 </div>
</div>

<div role="button" class="project" onclick="location.href='https://github.com/Grelot/diabetesGenetics--COAT'">
<img align="center" width="136rem" height="32rem" src="inserm.png">
<div class="title">Report bad quality region of coding sequences from genome sequencing data</div>
<div class="description">I developped a software able to detect human genomic variations with low coverage. Graphical Interface. </div>
<div class="skills"> (Illumina paired-end, samtools, bedtools, variation annotation, python, mySQL, qt4)</div>
<hr width="31%"> 
<div class="duration"> Jan 2016 - Jul 2016 </div>
</div>

<div role="button" class="project" onclick="location.href='https://www.dsimb.inserm.fr/orion/'">
<img align="center" width="136rem" height="32rem" src="inserm.png">
<div class="title">Optimization of a method of fold recognition in protein structure</div>
<div class="description">I added a new algorithm to predict 3D-modelisation of protein structure at atomic resolution </div>
<div class="skills"> (PDB, pymol, C, C++, python, R, html, css )</div>
<hr width="31%"> 
<div class="duration"> Feb 2015 - Jun 2015 </div>
</div>

</div>


_______________________________________________________________________________
 

# Softwares

## Florimond Desprez Group

* My software developed for the company is **confidential**. Please contact me for more information.

## Public research

* **[WFGD](https://shiny.cefe.cnrs.fr/wfgd/)** (main contributor): interactive worldmap of fish genetic diversity.
* **[ANVAGE](https://github.com/Grelot/anvage)** (main contributor): ANnotation Variants GEnome is a python toolkit software to perform routine operations such as detecting synonymous genetic variants from VCF, GFF3 and FASTA genome files.
* **[Rgeogendiv](https://github.com/Grelot/rgeogendiv)** (main contributor): R package for downloading, preparing and aligning georeferenced DNA sequences on Genbank to calculate genetic diversity at different geographical scales 
* **[Workflow to process environmental DNA sequencing data](https://gitlab.mbb.univ-montp2.fr/edna)** (main contributor): this workflow is open-source and was co-developped by the CEFE (main invistigator) and the company SPYGEN (data and tests), including interactions IFREMER, ETH Zurich and the marine explorations of Monaco.
* **[Workflow to genotype reduced genome sequencing data](https://gitlab.mbb.univ-montp2.fr/reservebenefit/snakemake_stacks2)** (main contributor): this workflow processed over 3000 fish genomes in the context of the european project RESERVEBENEFIT in collaboration with Helmholtz-Zentrum für Ozeanforschung Kiel and Instituto Español de Oceanografía.
* **[Genbar2](https://github.com/Grelot/genbar2)** (main contributor): identify genetic boundaries between populations using individual spatial coordinates and genetic variants.
* **[DEMORT](https://pypi.org/project/demort/)** (main contributor): a DEmultiplexing MOnitoring Report Tool
* **[EXAM](https://sourceforge.net/projects/exam-exome-analysis-and-mining/)**(contributor): a whole exome sequencing analysis and its graphical interface
* **[ORION](http://www.dsimb.inserm.fr/ORION/)** (contributor): a sensivitive method for protein template detection


_______________________________________________________________________________



# Publications

*Full list of publications on [Google Scholar](https://scholar.google.com/citations?hl=en&user=hj1ClrsAAAAJ)*


> **The distribution of coastal fish eDNA sequences in the Anthropocene**
>
> *Laetitia Mathon, Virginie Marques, Stéphanie Manel, Camille Albouy, Marco Andrello, Emilie Boulanger, Julie Deter, Régis Hocdé, Fabien Leprieur, Tom B Letessier, Nicolas Loiseau, Eva Maire, Alice Valentini, Laurent Vigliola, Florian Baletaud, Sandra Bessudo, Tony Dejean, Nadia Faure, Pierre‐edouard Guerin, Meret Jucker, Jean‐baptiste Juhel, Kadarusman, Andrea Polanco F, Laurent Pouyaud, Dario Schwörer, Kirsten F Thompson, Marc Troussellier, Hagi Yulia Sugeha, Laure Velez, Xiaowei Zhang, Wenjun Zhong, Loïc Pellissier, David Mouillot*
> 
> Global Ecology and Biogeography. 2023 August 08. DOI: [10.1111/geb.13698](https://doi.org/10.1111/geb.13698)


> **Contrasting influence of seascape, space and marine reserves on genomic variation in multiple species**
>
> *Laura Benestan, Nicolas Loiseau, Pierre‐edouard Guérin, Angel Pérez‐Ruzafa, Aitor Forcada, Esther Arcas, Philippe Lenfant, Sandra Mallol, Raquel Goñi, Laure Velez, David Mouillot, Oscar Puebla, Stéphanie Manel*
>
> Ecography. 2023 January 01. DOI: [10.1111/ecog.06127](https://doi.org/10.1111/ecog.06127)


> **Evaluating bioinformatics pipelines for population‐level inference using environmental DNA**
>
> *Bastien Macé, Régis Hocdé, Virginie Marques, Pierre‐Edouard Guerin, Alice Valentini, Véronique Arnal, Loïc Pellissier, Stéphanie Manel*
>
> Environmental DNA. 2022 February 08. DOI: [10.1002/edn3.269](https://doi.org/10.1002/edn3.269)


> **Cross-ocean patterns and processes in fish biodiversity on coral reefs through the lens of eDNA metabarcoding**
>
> *Laetitia Mathon, Virginie Marques, David Mouillot, Camille Albouy, Marco Andrello, Florian Baletaud, Giomar H Borrero-Pérez, Tony Dejean, Graham J Edgar, Jonathan Grondin, Pierre-Edouard Guerin, Régis Hocdé, Jean-Baptiste Juhel, Kadarusman, Eva Maire, Gael Mariani, Matthew Mclean, Andrea Polanco F, Laurent Pouyaud, Rick D Stuart-Smith, Hagi Yulia Sugeha, Alice Valentini, Laurent Vigliola, Indra B Vimono, Loïc Pellissier, Stéphanie Manel*
>
> Proceedings of the Royal Society B. 2022 April 20. DOI: [10.1098/rspb.2022.0162](https://doi.org/10.1098/rspb.2022.0162)


> **Climate differently influences the genomic patterns of two sympatric marine fish species**
>
> *Emilie Boulanger, Laura Benestan, Pierre‐Edouard Guerin, Alicia Dalongeville, David Mouillot, Stéphanie Manel*
> 
> Journal of Animal Ecology. 2021 October 30. DOI: [10.1111/1365-2656.13623]( https://doi.org/10.1111/1365-2656.13623)


> **Use of environmental DNA in assessment of fish functional and phylogenetic diversity**
>
> *Virginie Marques, Paul Castagne, Andréa Polanco Fernandez, Giomar Helena Borrero-Perez, Regis Hocde,  Pierre-Edouard Guerin, Jean-Baptiste Juhel, Laure Velez, Nicolas Loiseau, Tom Bech Letessier, Sandra Bessudo, Alice Valentini, Tony Dejean, David Mouillot, Loïc Pellissier, Sébastien Villéger*
>
> Conservation Biology. 2021 July 18. DOI: [10.1111/cobi.13802](https://doi.org/10.1111/cobi.13802)

> **Restricted dispersal in a sea of gene flow**
>
> *Laura Benestan, Nicolas Loiseau, Pierre-Edouard Guerin, Katarina Fietz, Elena Trofimenko, Siren Rühs, Willi Rath, Arne Biastoch, Angel Perez-Ruzafa, Pilar Baixauli, Aitor Forcada, Philippe  Lenfant, Sandra Mallol, Rachel Goni, Laure Velez, Marc Höppner, Stuart Kininmonth, David Mouillot, Oscar Puebla, Stephanie Manel*
>
> Proceedings of the Royal Society B. 2021 May 18. DOI: [10.1098/rspb.2021.0458](https://doi.org/10.1098/rspb.2021.0458)


> **Benchmarking bioinformatic tools for fast and accurate eDNA metabarcoding species identification**
>
> *Laetitia Mathon, Alice Valentini, Pierre-Edouard Guerin, Eric Normandeau, Cyril Noel, Clément Lionnet, Emilie Boulanger, Wilfried Thuiller, Louis Bernatchez, David Mouillot, Tony Dejean, Stephanie Manel*
>
> Molecular Ecology Resources. 2021 May 17. DOI: [10.1111/1755-0998.13430](https://doi.org/10.1111/1755-0998.13430)


> **Blind assessment of vertebrate taxonomic diversity across spatial scales by clustering environmental DNA metabarcoding sequences**
>
> *Virginie Marques, Pierre‐Edouard Guerin, Mathieu Rocle, Alice Valentini, Stephanie Manel, David Mouillot, Tony Dejean*
>
> Ecography. 2020 Aug 04. DOI: [10.1111/ecog.05049](https://doi.org/10.1111/ecog.05049)


> **New genomic ressources for three exploited Mediterranean fishes**
>
> *Katharina Fietz, Elena Trofimenko, Pierre-Edouard Guerin, Veronique Arnal, Montserrat Torres-Oliva, Stephane Lobreaux,Angel Perez-Ruzafa, Stephanie Manel, Oscar Puebla*
>
> Genomics. 2020 July 03. DOI: [10.1016/j.ygeno.2020.06.041](https://doi.org/10.1016/j.ygeno.2020.06.041)


> **Global determinants of freshwater and marine fish genetic diversity**
>
> *Stephanie Manel, Pierre-Edouard Guerin, David Mouillot, Simon Blanchet, Laure Velez, Camille Albouy & Loic Pellissier*
>
> Nature communications. 2020 Feb 10. DOI: [10.1038/s41467-020-14409-7](https://doi.org/10.1038/s41467-020-14409-7)


> **Predicting genotype environmental range from genome–environment associations**
>
> *Stephanie Manel, Marco Andrello, Karine Henry, Daphne Verdelet, Aude Darracq, Pierre‐Edouard Guerin, Bruno Desprez, Pierre Devaux*
>
> Molecular Ecology. 2018 May 17. DOI: [10.1111/mec.14723](https://doi.org/10.1111/mec.14723)


> **ORION : a web server for protein fold recognition and structure prediction using evolutionary hybrid profiles**
>
> *Yassine Ghouzam, Guillaume Postic, Pierre-Edouard Guerin, Alexandre G. de Brevern & Jean-Christophe Gelly* 
>
> Scientific Reports. 2016 Jun 20. DOI: [10.1038/srep28268](https://doi.org/10.1038/srep28268)


_______________________________________________________________________________

# Other activities


* My scientific [blog](https://guerinpe.com/articles/)
* Member of the association of Young French Bioinformaticians [Jebif](https://jebif.fr/en/)
* Author on french bioinformatics participative blog [bioinfo-fr](https://bioinfo-fr.net/author/pierre-edouard-guerin)
* My [data science skills demo](https://github.com/Grelot/machine_learning_demo) *(linear regression, random forest, neural network, bayesian, deep learning)* using python ecosystem for machine learning *(pandas, numpy, tensorflow, scikit-learn)* training on [kaggle](https://www.kaggle.com/pierreedouardguerin).







