+++
template = "page.html"
title = "Curriculum Vitae"
date =  2019-08-20
draft = false
tags = ["resume"]
+++




My name is Pierre-Edouard **GUERIN**. I'm currently working as a computational biologist in [CEFE](https://www.cefe.cnrs.fr) research team at [CNRS](http://www.cnrs.fr/languedoc-roussillon/03com-labos/03-2-lab-site/02-lab-site-montp.htm) of Montpellier, France.

_______________________________________________________________________________

<img align="right" width="48rem" height="96rem" src="paris7.png">

# Education



**2016: Master's degree of Bioinformatics**, Université Paris Diderot Paris 7, France



# Experience
<a href="https://www.ephe.fr/"><img align="right" width="66rem" height="62rem" src="ephe.png"></a>
<a href="https://www.cnrs.fr/"><img align="right" width="48rem" height="48rem" src="cnrs.png"></a>
**February 2017 to present: Computational biologist**  
Center of Functional Ecology and Evolution, Montpellier, France  
- Landscape genomics approach to assess genetic populations of mediteranean fishes for the european mission RESERVEBENEFIT  
- Program to process *metabarcoding* marine eDNA data for the world oceanographic mission of Monaco
- Genome assembly 

<a href="https://www.inserm.fr/"><img align="right" width="100rem" height="24rem" src="inserm.png"></a>
**January 2016 to august 2016: Software developer** (Master internship)
Genetics of Diabetes, Paris, FR  
- Human genomes DNA high throughput sequencing data analysis  
- Program to detect and annotate rare genetic variants  
- Graphical interface  


<a href="https://www.inserm.fr/"><img align="right" width="100rem" height="24rem" src="inserm.png"></a>
**February 2015 to June 2015: Computational biologist** (Master internship)  
- Dynamics of Structures and Interactions of Macromolecules in Biology Paris, FR  
- 3D-modelisation of protein structure at atomic resolution  
- Optimization of a method of fold recognition in protein structure  
- New algorithm to predict protein structure  

# Projects

## Metabarcoding


> <details><summary>One of the most promising genetic techniques for improving biodiversity assessments is the metabarcoding of environmental DNA. I did a state of the Art of available methods and developed serveral workflows to process and manage metabarcoding data from the Monaco Marine Scientific Exploration.</summary>
><p>
>
>
>
>### Overview: the necessity to assess marine biodiversity
> Marine environments, both coastal and offshore, are being severely impacted by traditional and emerging human activities. This is translated into habitat losses, pollution and overexploitation which treats marine >biodiversity. It compromises the sustainability of marine ecosystems and services.
>
>As a response to the environmental degradation, initiatives aims to protect marine ecosystems. Development of reliable marine biodiversity assessment methods is necessary. One of the most promising genetic techniques for >improving biodiversity assessments is the **metabarcoding** of environmental DNA.
>
>### What is Metabarcoding
>
>Indeed, all organisms shed cells containing DNA in their environment, as intra or extra-cellular material for up to a few days. The amplification and high-throughput eDNA sequencing followed by bioinformatic analyses >produces a list of sequences with the ultimate goal to assess species diversity in a given site.
>
>### Assess marine biodiversity all over the world with metabarcoding
>
>eDNA samples were collected by [Monaco Scientific Exploration Yersin](https://fr.wikipedia.org/wiki/Yersin_(navire_oc%C3%A9anographique)) in Guadeloupe, Lengguru, Malpelo Fakarava and Mediteranean sea. Sequencing were performed by [SPYGEN company](http://www.spygen.com/) and I was in charge of the bioinformatics processing of sequencing data.
>
>### My contribution as a computational biologist
>
>I did a state of the Art of available methods and developed serveral workflows to process **metabarcoding** data in order to assess marine biodiversity all over the world. Source codes are available as git repositories on the [Montpellier server dedicated to eDNA analysis](https://gitlab.mbb.univ-montp2.fr/edna).
>
>
></p>
></details>




## Genotyping by sequencing


> Montpellier, FR
> <br><details><summary> </summary>
><p>
>
>### Overview:
>
>
>
></p>
></details>


## Genome assembly

> Montpellier, FR
> <br> *02.2017 - current*
> <br><details><summary>s</summary>
><p>
>
>### Overview: the necessity to produce whole genome sequence
>
>### What is a genome ?
>
> Genome sequencing is figuring out the order of DNA nucleotides, or bases, in a genome—the order of As, Cs, Gs, and Ts that make up an organism's DNA. For instance, the human genome is made up of over 3 billion of these genetic letters.
>
>### Sequencing the genome
>
>
>### New genomes available for three species of mediterranean fishes
>
>
></p>
></details>


## Global determinants of freshwater and marine fish genetic diversity

## genbar2

## Predicting genotype environmental range from genome–environment associations






# Softwares

* **[Genbar2](https://github.com/Grelot/genbar2)**: identify genetic boundaries between populations using individual spatial coordinates and genetic variants
* **[DEMORT](https://pypi.org/project/demort/)**: a DEmultiplexing MOnitoring Report Tool
* **[EXAM](https://sourceforge.net/projects/exam-exome-analysis-and-mining/)**: a whole exome sequencing analysis and its graphical interface
* **[COAT](https://github.com/Grelot/diabetesGenetics--COAT)**: a COverage Analysis Tool for exome sequencing data
* **[ORION](http://www.dsimb.inserm.fr/ORION/)**: a sensivitve method for protein template detection


# Publications

* **New genomic ressources for three exploited Mediterranean fishes** (submitted to Molecular Ecology Ressources in november 2019)
* **Global determinants of freshwater and marine fish genetic diversity** (submitted to Nature Communications in februrary 2019)
* **Predicting genotype environmental range from genome–environment associations** ([Molecular Ecology in may 2018](https://doi.org/10.1111/mec.14723))
* **ORION : a web server for protein fold recognition and structure prediction using evolutionary hybrid profiles** ([Scientific Reports in june 1016](https://doi.org/10.1038/srep28268))


_______________________________________________________________________________

# Other activities


## Sciences

* My scientific [blog](https://guerinpe.com/articles/)
* Member of the association of Young French Bioinformaticians [Jebif](https://jebif.fr/en/)
* Author on french bioinformatics participative blog [bioinfo-fr](https://bioinfo-fr.net/author/pierre-edouard-guerin)



## Personnal projects

<img align="right" width="60px" height="60px" src="costa_walk.png">

* **[speckyman](https://github.com/Grelot/speckyman)**: a platform game developed in JavaScript
* **[fromdnatomusic](https://github.com/Grelot/fromdnatomusic)**: convert DNA sequences into MIDI track and listen to it
* **[Nos data ont du talent](https://www.youtube.com/channel/UCvjBNumU6EvJiiGfxqNfd7Q)**: a youtube channel dedicated to data visualisation



## Triathlon


I'm an active member of the [USN-MONTPELLIER](https://www.usn-montpellier.fr/usn-web/view/index.php) sportive club since 2017. Join us !

<img align="center" width="264rem" height="247rem" src="usnm.png">


______



